<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>

    <!-- font -->
    <!-- <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" /> -->
    <link rel="stylesheet" href="https://cdn.staticfile.org/font-awesome/4.7.0/css/font-awesome.css">

    <!-- AOS -->
    <!-- <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" /> -->

    <!-- css -->
    <link rel="stylesheet" href="../css/p_project_css.css">

    <!-- bootstrap -->
    <link rel="stylesheet" href="../css/p_main.css">
    <style>
        table,
td,
th {
  text-align: center;
  border: 1px solid #000;
  border-collapse: collapse;
}       
    </style>
</head>

<body>
    <div id="loader">
        <img src="../img/loader.gif" alt="Loading...">
    </div>
    <div id="toopp"></div>
    <div class="header">
        <div>
            <nav class="navbar navbar-expand-md">
                <a href="" class="navbar-brand"><img src="../img/logo.png" alt="" class="logo"></a>
                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navmenu">
                    <span class="navbar-toggler-icon"></span>
                </button>
                <div class="collapse navbar-collapse" id="navmenu">
                    <ul class="nav-list navbar-nav ms-auto">
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="../index.html">
                                    <div class="nav-link">Home</div>
                                </a>
                            </li>
                        </div>
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="../index.html">
                                    <div class="nav-link">Project</div>
                                </a>
                                <ul>
                                    <li><a href="../pages/Description.html">
                                            <div><img src="../img/icon2.png" alt="">Description</div>
                                        </a></li>
                                    <li><a href="../pages/Design.html">
                                            <div><img src="../img/icon2.png" alt="">Design</div>
                                        </a></li>
                                    <li><a href="../pages/Results.html">
                                            <div><img src="../img/icon2.png" alt="">Results</div>
                                        </a></li>
                                    <li><a href="../pages/Supplement.html">
                                            <div><img src="../img/icon2.png" alt="">Supplement</div>
                                        </a></li>
                                    <li><a href="../pages/Protocol.html">
                                            <div><img src="../img/icon2.png" alt="">Protocol</div>
                                        </a></li>
                                </ul>
                            </li>
                        </div>
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="../pages/notebook.html">
                                    <div class="nav-link">Lab Notebook</div>
                                </a>
                            </li>
                        </div>
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="#">
                                    <div class="nav-link">Team</div>
                                </a>
                                <ul>
                                    <li><a href="../pages/TEAM页.html">
                                        <div><img src="../img/icon2.png" alt="">Team</div>
                                    </a></li>
                                    <li><a href="#">
                                        <div><img src="../img/icon2.png" alt="">Collaboration</div>
                                    </a></li>
                                </ul>
                            </li>
                        </div>
                    </ul>
                </div>
            </nav>
        </div>

    </div>
    <div class="headPicture">
        <img src="../img/protocol.jpg" alt="" class="notebookImg">
        <img src="../img/protocol.png" alt="" class="notebookText3">
    </div>
    <div class="benti">
   <div class="page">
    <div class="section">
        <h1 id="b">Materials</h1>
        <p><strong style="font-size: 25px;">Biological materials</strong></p>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody><tr>
                <th>Names</th>
                <th>Relevant characteristics</th>
                <th>Source</th>
            </tr>
            <tr>
                <td>DH5α</td>
                <td>deoR endA1 gyrA96 hsdR17 (rk-mk+)recA1 relA1 supE44 thi-1 Δ(lacZYA-argF)U169 Φ80lacZ ΔM15F - λ -</td>
                <td>Vazyme</td>
            </tr>
            <tr>
                <td>BL21(DE3)</td>
                <td>F-ompThsdS(rB-mB-) galdcm (DE3)</td>
                <td>Vazyme</td>
            </tr>
            <tr>
                <td>S2060</td>
                <td>F’ proA+B+ Δ(lacIZY) zzf::Tn10 lacIQ1 PN25-tetR luxCDE Ppsp(AR2) lacZ luxR Plux groESL / endA1 recA1 galE15 galK16 nupG rpsL
                    ΔlacIZYA araD139 Δ(ara,leu)7697 mcrA Δ(mrr-hsdRMS-mcrBC) proBA::pir116 araE201 ΔrpoZ Δflu ΔcsgABCDEFG ΔpgaC λ–
                    </td>
                <td>Addgene1</td>
            </tr>
            <tr>
                <td>S2208</td>
                <td>F’ proA+B+ Δ(lacIZY) zzf::Tn10 lacIQ1 PN25-tetR luxCDE Ppsp(AR2) lacZ luxR Plux groESL / endA1 recA1 galE15 galK16 nupG rpsL
                    ΔlacIZYA araD139 Δ(ara,leu)7697 mcrA Δ(mrr-hsdRMS-mcrBC) proBA::pir116 araE201 ΔrpoZ Δflu ΔcsgABCDEFG ΔpgaC λ–
                    </td>
                <td>Addgene</td>
            </tr>
        </tbody></table>
        <p><strong style="font-size: 25px;">Plasmids</strong></p>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody><tr>
                <th>Names</th>
                <th>Relevant characteristics</th>
                <th>Purpose</th>
            </tr>
            <tr>
                <td>pQE-80L</td>
                <td>pBR322-KanR</td>
                <td>Expression vector and negative control</td>
            </tr>
            <tr>
                <td>pQE-80L-eGFP</td>
                <td>pQE-80L-P22-T-PfdeAR-eGFP</td>
                <td>positive control</td>
            </tr>
            <tr>
                <td>pQE-80L-fdeR</td>
                <td>pBR322-p22-fdeR-T-PfdeAR-eGFP</td>
                <td>Biosensor</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-A</td>
                <td>pBR322-p22-fdeR-Mutpre-L268W</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-B</td>
                <td>pBR322-p22-fdeR-Mutpre-H170W</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-C</td>
                <td>pBR322-p22-fdeR-Mutpre-H170Y</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-D</td>
                <td>pBR322-p22-fdeR-Mutpre-H170W+L268W</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-E</td>
                <td>pBR322-p22-fdeR-Mutpre-H170Y+L268W</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>AP1</td>
                <td>pSC101Cm-T7-gⅢ-eGFP</td>
                <td>positive selection</td>
            </tr>
            <tr>
                <td>AP2-neg</td>
                <td>pSC101Cm-T7-gⅢ-neg-RFP</td>
                <td>positive selection</td>
            </tr>
            <tr>
                <td>DP6<sup>2</sup></td>
                <td>A plasmid expresses mutagenic genes </td>
                <td>Expression of mutagenic genes </td>
            </tr>
            <tr>
                <td>pBT137-C<sup>2</sup></td>
                <td>M13 carrier section 
                    with replica origin
                    </td>
                <td>Construction of selection phage</td>
            </tr>
            <tr>
                <td>pBT137-D<sup>2</sup></td>
                <td>M13 carrier fraction 
                    containing gⅢ
                    </td>
                <td>Construction of selection phage</td>
            </tr>
            <tr>
                <td>AP-P-Mid2-T7RNAP</td>
                <td>Contains Narigenin riboswitch Mid2 and T7RNAP</td>
                <td>Construction of selection phage</td>
            </tr>
            <tr>
                <td>AP-P-High2-T7RNAP</td>
                <td>Contains Narigenin riboswitch High2 and T7RNAP</td>
                <td>Construction of selection phage</td>
            </tr>
            <tr>
                <td>pJC175e<sup>2</sup></td>
                <td>Provides low copy of gⅢ</td>
                <td>Construction of selection phage</td>
            </tr> 
        </tbody></table>
        <p><strong style="font-size: 25px;">Primers</strong></p>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody>
                <tr><th>No.</th>
                <th>Name</th>
                <th>Sequence (5’ -> 3’)</th>
                <th>Function</th></tr>
                <tr>
                <td>1</td>
                <td>H170Y-1-F</td>
                <td>5'-gcagacGTAccgttcgcggaagacctct-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>2</td>
                <td>SITE-1-R</td>
                <td>5'-cgcatatggtgcactctcagtaca-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>3</td>
                <td>H170Y-2-R</td>
                <td>5'-gaacggTACgtctgcgtggtctggcgc-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>4</td>
                <td>SITE-2-F</td>
                <td>5'-tgtactgagagtgcaccatatgcg-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>5</td>
                <td>L268W-1-F</td>
                <td>5'-ctcgccCCAcgacagcggactctccttgatc-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>6</td>
                <td>SITE-1-R</td>
                <td>5'-cgcatatggtgcactctcagtaca-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>7</td>
                <td>L268W-2-R</td>
                <td>5'-ctgtcgTGGggcgagatgcggcagatgat-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>8</td>
                <td>SITE-2-F</td>
                <td>5'-tgtactgagagtgcaccatatgcg-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>9</td>
                <td>H170W-1-F</td>
                <td>5'-gcagacCCAccgttcgcggaagacctct-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>10</td>
                <td>SITE-1-R</td>
                <td>5'-cgcatatggtgcactctcagtaca-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>11</td>
                <td>H170W-2-R</td>
                <td>5'-gaacggTGGgtctgcgtggtctggcgc-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>12</td>
                <td>SITE-2-F</td>
                <td>5'-tgtactgagagtgcaccatatgcg-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>13</td>
                <td>AP2-neg- GⅢneg-F</td>
                <td>5'-TTAAGACTCCTTATTACGCAGTATGTT-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>14</td>
                <td>AP2-neg- GⅢneg-R</td>
                <td>5'-GGTGGCTCTTCCCAAATGGCTCAAGTCG-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>15</td>
                <td>RFP-F</td>
                <td>5'-TGCTAGAACTGGCATGATTAAGCACCGGTGGAGTGAC-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>16</td>
                <td>RFP-R</td>
                <td>5'-GTAATAAGGAGTCTTAAATGGCGAGTAGCGAAGACGT-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>17</td>
                <td>AP2-neg-vector-F</td>
                <td>5'-TTTGGGAAGAGCCACCACCGGAACCG-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>18</td>
                <td>AP2-neg-
                    vector-R
                    </td>
                <td>5'-TCATGCCAGTTCTAGCATAACCCC-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>19</td>
                <td>AP2-pos-vector-F</td>
                <td>5'-CTCACCATTTAAGACTCCTTATTACGCAGTATGTTAG-3'</td>
                <td>Construction of AP2-pos</td>
            </tr>
            <tr>
                <td>20</td>
                <td>AP2-pos-vector-R</td>
                <td>5'-ACAAGTAATCATGCCAGTTCTAGCATAACC-3'</td>
                <td>Construction of AP2-pos</td>
            </tr>
            <tr>
                <td>21</td>
                <td>AP2-pos-eGFP-F</td>
                <td>5'-AGTCTTAAATGGTGAGCAAGGGCGAGGAG-3'</td>
                <td>Construction of AP2-pos</td>
            </tr>
            <tr>
                <td>22</td>
                <td>AP2-pos-eGFP-R</td>
                <td>5'-TGGCATGATTACTTGTACAGCTCGTCCATGCC-3'</td>
                <td>Construction of AP2-pos</td>
            </tr>
            <tr>
                <td>23</td>
                <td>pQE-pathway-vector-F</td>
                <td>5'-TAAGGCAGTTATTGGTGCCCTTA-3'</td>
                <td>Construction of pQE-pathway</td>
            </tr>
            <tr>
                <td>24</td>
                <td>pQE-pathway-vector-R</td>
                <td>5'-TTCTCGAGGTGAAGACGAAAGG-3'</td>
                <td>Construction of pQE-pathway</td>
            </tr>
            <tr>
                <td>25</td>
                <td>pQE-pathway-fragment-F</td>
                <td>5'-TCTTCACCTCGAGAATCAGCAGATGCCTGGCAGC-3'</td>
                <td>Construction of pQE-pathway</td>
            </tr>
            <tr>
                <td>26</td>
                <td>pQE-pathway-fragment-R</td>
                <td>5'-CCAATAACTGCCTTAAAAAAATTACTTGTACAGCTCGTCCATGC-3'</td>
                <td>Construction of pQE-pathway</td>
            </tr>
            <tr>
                <td>27</td>
                <td>pQE-SplitC-RBS-F</td>
                <td>5'-ACTAGACTACACTAAAAAGAGGACTATTCATCCGCTGCATGGCCCC-3'</td>
                <td>Substitution of RBS of eGFP</td>
            </tr>
            <tr>
                <td>28</td>
                <td>pQE-SplitC-RBS-R</td>
                <td>5'-CGCATATGGTGCACTCTCAGTACAATCTG-3'</td>
                <td>Substitution of RBS of eGFP</td>
            </tr>
            <tr>
                <td>29</td>
                <td>pQE-SplitD-RBS-F</td>
                <td>5'-TGTACTGAGAGTGCACCATATGCG-3'</td>
                <td>Substitution of RBS of eGFP</td>
            </tr>
            <tr>
                <td>30</td>
                <td>pQE-SplitD-RBS-R</td>
                <td>5'-TCTTTTTAGTGTAGTCTAGTAATCCCGTTGATATTGATCAAATGGATTGTTTTG-3'</td>
                <td>Substitution of RBS of eGFP</td>
            </tr>
            <tr>
                <td>31</td>
                <td>Mid2-random-F</td>
                <td>5'-CGATGCTGCTTATCGAAGTCTGA-3'</td>
                <td>Nested PCR to obtain the fragment of Mid2</td>
            </tr>
            <tr>
                <td>32</td>
                <td>Mid2-random-R</td>
                <td>5'-TGCAGAACGCGCCTGTTTATC-3'</td>
                <td>Nested PCR to obtain the fragment of Mid2</td>
            </tr>
            <tr>
                <td>33</td>
                <td>High2-random-F</td>
                <td>5'-AGGCGCGTTCTGCATTAGC-3'</td>
                <td>Nested PCR to obtain the fragment of High2</td>
            </tr>
            <tr>
                <td>34</td>
                <td>High-random-R</td>
                <td>5'-GCTGTAACAAGTTGTCTCAGGTGTTC-3'</td>
                <td>Nested PCR to obtain the fragment of High2</td>
            </tr>
            <tr>
                <td>35</td>
                <td>overlap-Mid2&High2-F</td>
                <td>5'-TACTACATAAATCAAACAGGGACGACGATGACACGAGA-3'</td>
                <td> Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>36</td>
                <td>overlap-Mid2-R</td>
                <td>5'-GTTAATCGTGTTCATAGATGCTCCTTCACTCGCATT-3'</td>
                <td>Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>37</td>
                <td>overlap-High2-R</td>
                <td>5'-GTTAATCGTGTTCATAGATGCTCCTTATTCCCCCC-3'</td>
                <td> Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>38</td>
                <td>overlap-T7RNAP-F</td>
                <td>5'-AGGAGCATCTATGAACACGATTAACATCGCTAAGA-3'</td>
                <td> Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>39</td>
                <td>overlap-T7RNAP-R</td>
                <td>5'-GGCATGAGCTCTTCAGCCTTACGCGAACGCGAAGTCC-3'</td>
                <td> Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>40</td>
                <td>overlap-AP-P-V-F</td>
                <td>5'-GTTCGCGTAAGGCTGAAGAGCTCATGCCAG-3'</td>
                <td>Construction of the plasmid of AP-P-Mid2-T7RNAP and AP-P-High2-T7RNAP</td>
            </tr>
            <tr>
                <td>41</td>
                <td>overlap-AP-P-V-R</td>
                <td>5'-GTCGTCCCTGTTTGATTTATGTAGTAGACTAGAAGAGC-3'</td>
                <td>Construction of the plasmid of AP-P-Mid2-T7RNAP and AP-P-High2-T7RNAP</td>
            </tr>
            <tr>
                <td>42</td>
                <td>ep-fdeR-F</td>
                <td>5'-ACCTCGAGAACGATGCTGCTTATCGAAGTCTGA-3'</td>
                <td>Error-prone PCR to obtain fdeR</td>
            </tr>
            <tr>
                <td>43</td>
                <td>ep-fdeR-R</td>
                <td>5'-CTCGAATTCGGAGGAAACAAAG-3'</td>
                <td>Error-prone PCR to obtain fdeR</td>
            </tr>
            <tr>
                <td>44</td>
                <td>ep-vec-F</td>
                <td>5'-TGTTTCCTCCGAATTCGAGGT-3'</td>
                <td>Error-prone PCR to obtain pQE-80L used for vector</td>
            </tr>
            <tr>
                <td>45</td>
                <td>ep-vec-R</td>
                <td>5'-CATCGTTCTCGAGGTTCGTGATACGCCTATTTTTATAGGTTA-3'</td>
                <td>Error-prone PCR to obtain pQE-80L used for vector</td>
            </tr>
        </tbody></table>
        <p><strong style="font-size: 25px;">Reagents</strong></p>
        <p>YEAST EXTRACT (OXOID, cat. no. LP0021) TRYPTONE (OXOID,cat. no. LP0042) NaCl (HUSHI,cat. no. 10019318)
            X-Gal (Solarbio, cat. no. R0404) <br>
            Agar (Solarbio, cat. no. A8190) <br>
            Ampicillin (Solarbio, cat. no. A7094) <br>
            Streptomycin (Solarbio, cat. no. S8290) <br>
            Kanamycin (Solarbio, cat. no. K8020) <br>
            SapⅠ (New England BioLabs, cat. no. R0569S) <br>
            T4 DNA ligase (New England BioLabs, cat. no. M0202V). <br>
            IPTG (Isopropyl β-D- Thiogalactopyranoside ) (Solarbio, cat. no. I8070) <br>
            LB medium <br>
            FastPure Gel DNA (Nanjing Vazyme, cat. DC301-01) <br>
            FastPure Plasmid Mini kit (Nanjing Vazyme, cat. DC201-01) <br>
            DH5α Competent cell (Nanjing Vazyme, cat. C502-02) <br>
            ClonExpress MultiS One Step Cloning Kit (Nanjing Vazyme, cat. C113-01) <br>
            ClonExpress II One Step Cloning Kit (Nanjing Vazyme, cat. C112-01) <br>
            2×YT liquid media <br>
            (Add 2×YT media powder to a final concentration of 31 g/L in water. Mix to dissolve and then autoclave to sterilize at 121.0 °C. Store at room temperature indefinitely.) <br>
            2×YT agar <br>
            (Add 2×YT media powder and agar to a final concentration of 31 g/L and 1.5% (wtol), respectively, in water. Autoclave to sterilize at 121.0 °C. Store at room temperature in the dark for up to 6 months.) <br>
            2×YT top agar <br>
            (Melt 2×YT agar using a microwave until completely liquid. Dilute hot 2×YT agar with room temperature 2×YT liquid media to a final agar concentration of 0.5% or 0.6%(wtol) and mix thoroughly.) <br>
            LB liquid media <br>
            (Add YEAST EXTRACT, TRYPTONE, NaCl to a final concentration of 5 g/L,10 g/L,10 g/L in water, respectively.) <br>
            LB agar <br>
            (Add YEAST EXTRACT, TRYPTONE, NaCl, agar to a final concentration of 5 g/L, 10g/L, 10g/L, 1.5% (wtol), respectively, in water.) <br>
            1×PBS(Solarbio, cat. P1020) <br>
            10×PBS(Solarbio, cat. P1022) <br>
            </p>
            <p><strong style="font-size: 25px;">Equipment</strong></p>
            <p>Biological shaker (Shanghai Zhichu Instruments, cat. no. ZHTY-50ES) <br>
                Benchtop centrifuge (Thermo Fisher Scientific, cat. no. 75007201) <br>
                Electrophoresis(Beijing Liuyi, cat. no. DYY-6C) <br>
                Absorbance Microplate Reader(TENCAN, cat. no.Tecan Infinite M200 NanoQuant) <br>
                Isolated thermostatic incubator (Shanghai SENXIN , cat. no. GNP-9080) <br>
                Gene Explorer(Bioer technology, cat. TC-96/G/H(b)B) <br>
                13 mm, 0.22-μm PVDF syringe filter(0.22 µm; Merck Millipore Ltd., cat. no. PR05538) <br>
                Gel imaging system(CLiN Qingxiang, cat. no. GenoSens2100) <br>
                </p>
    </div>
    <div class="section">
        <h1 id="p">Methods</h1>
        <p><strong style="font-size: 25px;">Preparation</strong></p>
        <p><strong >Mention of plasmid</strong> <br>
            According to the kit instructions for Nanjing Vazyme. <br>
            <strong >PCR fragment and vector</strong> <br>
            The obtained plasmids are subjected to PCR using different programs and primers. <br>
PCR fragment and vector respectively with corresponding primers. <br>
After agarose gel electrophoresis, we extract DNA from the gel. The products are stored at -20 ℃ for subsequent use. <br>
        </p>
        <p><strong style="font-size: 25px;">Setting mutation conditions</strong></p>
        <p><strong >Error-prone PCR</strong>
            Use StarMut Random Mutagenesis Kit. Multiple small system (20μL) amplification pre-experiments were carried out, setting a series of StarMut Enhancer gradients. According to the PCR results and the mutation rate in the instructions, Enhancer volume (4μL) was determined.
            <strong >Site-directed mutagenesis</strong>
            With the help of two bioinformatic tools: AlphaFold2 and AutoDock 4.2.6, we spot three mutational residues: L268W、H170Y、H170W. Several pairs of degenerate primer were designed to randomly introduce mutations in fdeR.
        </p>
        <p><strong style="font-size: 25px;">Mutation</strong></p> 
        <p><strong >Error-prone PCR of fdeR</strong>
            Using the error-prone PCR kit (Gene-star company) in a total volume of 25 μL using template 10 ng of pQE-80L-fdeR plasmid as template , 10μL 2×StarMut Random PCR Mix , 10 mM Gen 2-F, 10 μM Gen 2-R each, Star Mut enhancer 6 μL, sterile water was made up to 25 μL, 53.1℃) or error-prone PCR on fdeR fragments. Subsequently, the above fdeR gene fragments were sequenced by Sanger Sequence. Then based on the sequencing results, the number of mutated bases per thousand bases was calculated. Meanwhile, the linearized vector and the above fragments were assembled by ClonExpress II One Step Cloning Kit (Vazyme, Nanjing, China). E. coli BL21 was used for subsequent chemical transformation.
            <strong >Site-Directed Mutation of FdeR</strong>
            For construction of the single and double mutants of fdeR, the template the pQE-80L-fdeR plasmid was used along with a suitable combination of primers(Table S3).Three pairs of primers were designed to introduce three mutations (H170W,H170Y,L268W) on two sites in FdeR. The whole pQE-80L-fdeR plasmid was divided into two or three parts and then amplified by two or three pairs of primers(Table S3). The consequent two or three fragments with introduced mutations were combined with ClonExpress II One Step Cloning Kit or ClonExpress MultiS One Step Cloning Kit (Vazyme, Nanjing, China), which is a cloning kit based on homologous recombination. E.coli DH5α was used for subsequent chemical transformation. The plasmids from individual transformants were used to DNA sequence to confirm that correct mutation was obtained. Than, plasmids with correct mutations were transformed into E.coli BL21. Randomly select individual colonies for fluorescence screening.
            <strong >Site-Directed Mutation of FdeR</strong>
        </p>
        <p><strong style="font-size: 25px;">Bioinformatics analysis</strong></p> 
        <p><strong>Molecular Docking</strong> <br>
            We used AutoDock 4.2.6 28 for protein-small molecule docking. We obtained the 3D structure of the protein receptor using AlphaFold2 (Figure S10), and then performed a series of operations such as adding hydrogen, merging non-polar hydrogen, assigning Gasteiger charges, and storing it as a PDBQT format file. We obtained the planar conformation of liquiritigenin from PubChem, and then used OpenBabel to perform conformational conversion to form the optimal three-dimensional conformation required for docking and store it as a mol2 format file (Figure S9). Using AutoDockTools (ADT), we performed operations such as adding hydrogen and merging non-polar hydrogen atoms on liquiritigenin, and calculated the Gasteiger-Huiekel charges. The ligand was also stored as a PDBQT format file. <br>
            We chose the LGA algorithm for energy optimization, and set the relevant parameters of the Autogrid4 docking box and the Autodock4 docking program, and finally obtained a variety of docking conformations clustered by binding energy and RMSD - the computer selected the docking result with the lowest binding energy, and then used this docking conformation as a reference to compare other docking results. Compared with the control, those with a mean square deviation and structural difference of less than 0.5 Å were grouped into one group (cluster), and those with a difference of more than 0.5 Å were divided into another group. The different docking results in the same group were also sorted by binding energy and compared with the first one in the group. Those with a difference of less than 0.5 Å were left in the group, and those with a difference of more than 0.5 Å were re-grouped, and so on. <br>
            We used Discovery Studio to perform DNA 3D modeling of the fdeR gene promoter, and then used the protein as the receptor and the promoter as the ligand to perform DNA-protein docking on pyDockDNA. Then, according to the scoring ranking provided by pyDockDNA, we comprehensively judged and selected the docking results. The final screened conformation was analyzed and plotted using Pymol. <br>
            <strong>Prediction of single mutation sites</strong> <br>
            We use the software of Discovery Studio 2021 Client to predict the mutation sites. The receptor protein was applied CHARMm force field. Mutation stability was processed on single site using Calculate Mutation Energy (Binding) protocol of Discovery Studio 2021 Client.  <br>
            Calculate Mutation Energy (Binding) protocol evaluates the effect of single-point mutations on protein stability. It performs amino-acid scanning mutagenesis on a set of selected amino-acid residues by mutating each of them to one or more specified amino-acid types. The mutation energy is the change in the protein’s stability caused by a mutation. It is measured by comparing how much energy the protein needs to fold in its original and mutated forms. The more energy the protein needs to fold, the less stable it is. <br>
            <strong>Molecular dynamics (MD) simulations</strong> <br>
            The molecular dynamics (MD) simulations were carried out by GROMACS 2020.3 software. The amber99sb-ildn force field and the general Amber force field (GAFF) were used to generate the parameter and topology of proteins and ligands, respectively. The simulation box size was optimized with the distance between each atom of the protein and the box greater than 1.0 nm. Then, fill the box with water molecules based on a density of 1. To make the simulation system electrically neutral, the water molecules were replaced with Cl- and Na+ ions. Following the steepest descent method, energy optimization of 5.0×104 steps was performed to minimize the energy consumption of the entire system, and finally to reduce the unreasonable contact or atom overlap in the entire system. After energy minimization, first-phase equilibration was performed with the NVT ensemble at 300 K for 100 ps to stabilize the temperature of the system. Second-phase equilibration was simulated with the NPT ensemble at 1 bar and 100 ps. The primary objective of the simulation is to optimize the interaction between the target protein and the solvent and ions so that the simulation system is fully pre-equilibrated. All MD simulations were performed for 50 ns under an isothermal and isostatic ensemble with a temperature of 300 K and a pressure of 1 atmosphere. The temperature and pressure were controlled by the V-rescale and Parrinello-Rahman methods, respectively, and the temperature and pressure coupling constants were 0.1 and 0.5 ps, respectively. Lennard-Jones function was used to calculate the Van der Waals force, and the nonbond truncation distance was set to 1.4 nm. The bond length of all atoms was constrained by the LINCS algorithm. The long-range electrostatic interaction was calculated by the Particle Mesh-Ewald method with the Fourier spacing 0.16 nm.
        </p>
        <p><strong style="font-size: 25px;">Transformation</strong></p> 
        <p>Competent cells were thawed on ice. Pre-cooled plasmid mixture was added (each plasmid should < 100 ng; Each transformation of up to 3 plasmids), fully mixed after 30min on ice, 42℃ water bath, heat shock 90s(For S2060 and S2208 strains, heat shock 75s). Put back on ice and stand for 2-3min; Add 500 microliters of LB liquid medium, shake at 37℃, 200-300rpm/min, 1h; The bacterial solution was centrifuged at 5000rpm for 3min, the appropriate supernatant was removed, and then resuspended and coated in LB solid medium(For S2060 and S2208 strains, use 2×YT solid medium) containing appropriate antibiotics and 1.5% agar at 37℃ for 10-12h.</p>

        
        <p><strong style="font-size: 25px;">Screening</strong></p> 
        <p><strong>Fluorescence activated cell sorting (FACS) and 96-well plates screening</strong><br>
            Cultures of E. coli BL21 with empty pQE-80L plasmids (Negative control) or transformed with plasmids containing eGFP reporter (Positive control). Library of FdeR mutants were grown overnight in 5 ml LB medium supplemented with 50 mg/mL kanamycin which grown on a biological shaker for 12 h at 220 rpm and 37℃. Cultures were diluted (1:100) into fresh media, grown until OD600~0.6 and induced with 0.5 mM liquiritigenin for 12 h. <br>
            The culture was diluted 10-fold in fresh liquid LB medium and analysed using FACS (BD FACSAria III, Institute of Soil Science, Chinese Academy of Sciences). A histogram plot was generated by sorting cells with empty pQE-80L plasmids and defining a negative gate as negative control. Meanwhile, using cells with plasmids containing eGFP reporter to designate a positive gate as a postive control. Two rounds of FACS counter-selection were constructed with liquiritigeni induction or naringenin induction. <br>
            In the 1st round, positive sorting was performed, collecting the population on the top 5%~10% of the histogram plot under the positive gate. At least 2.4×107 events were analysed by the flow cytometer, which can cover entire experimental libraries( 5.6× 106). The collected cells were added into 5ml liquid LB medium supplemented with kanamycin to grow at biological shaker 220 rpm, 37℃, overnight. Cultures were diluted (1:100) into fresh LB medium, grown until OD600~0.6 and induced with 0.5 mM naringenin for 12 h as the samples of 2nd round. <br>
            In the 2nd round, negative sorting was performed, collecting the population on the bottom 5%~10% within the negative gate. At least 4.1×106 events were passed through the flow cytometer corresponding to 10-fold the initial library size. The sorted cells were plated on LB agar meium supplemented with 50 mg/mL Kanamycin overnight. Colonies were used for subsequent 96-well plates screening. <br>
            <strong>96 deep-well plates screening</strong> <br>
            The bacterial solution in 96 deep-well plate was centrifuged at 4000 rpm for 8 min after the induction culture completed, and then discard the supernatant. 1000 μL cold PBS buffer per well was pipetted and resuspended, and 200μl was taken into a 96-well fluorescent plate. Use a microplate reader to measure the fluorescence at Ex = 482 nm/Em = 524 nm and the absorbance at OD600. <br>

        </p>
        <p><strong style="font-size: 25px;">Characterization and compensation test of biosensor</strong> <br>
            A concentration gradient of liquiritigenin ranging from 0 to 1 mM was introduced into 1 mL of bacterial culture containing kanamycin in a 96 deep-well plate, supplemented with 0.5 mM naringenin. The well-characterized variants were introduced into 5 mL of LB medium with kanamycin and incubated overnight at 37°C and 220 rpm on a shaker until reaching saturation. Plasmids were extracted using the FastPure Plasmid Mini Kit (Vazyme, Nanjing, China) and subsequently retransformed into BL21(DE3) competent cells (Vazyme, Nanjing, China). These cells were inoculated into 5 mL of kanamycin-supplemented LB medium and grown overnight on a Biological Shaker at 220 rpm and 37°C. Following cell resuspension, fluorescence intensity was measured using a microplate reader. <br>
            <strong style="font-size: 25px;">Characterization and compensation test of biosensor</strong> <br>
            When measuring fluorescence, we used 1×PBS to correct for background fluorescence and optical density of the PBS (FPPBS and ODPBS, respectively) for each imposed naringenin or liquiritigenin concentration. The normalized fluorescence for optical density was calculated as follows:  <br>
            The resulting mean each concentration of naringenin or liquiritigenin, are expressed in arbitrary units (a.u.) and were fitted with the Hill function as follow using the weighted nonlinear least-squares algorithm (curve_fit, SciPy, Levenberg−Marquardt algorithm):
        </p>
        <div style="display: flex;justify-content: center; "><img class="gs"  src="../img/gs1.png" alt="" style="width: 40%;"></div>
        <p>with C: the concentration of naringenin in the growth medium (mg/L); a: the basal normalized fluorescent signal (leaky expression, au); k: the relative maximum normalized fluorescent signal (au); n: the Hill coefficient (cooperativity, sigmoid character); KM: the Hill constant (half-maximal naringenin concentration, mg/L).  <br>
            We apply a Noise parameter as an indicator for the overall mean error on the response of FdeR. The average relative error was calculated across the imposed naringenin or liquiritigenin concentrations, as follows:</p>
        <div style="display: flex;justify-content: center; "><img class="gs"  src="../img/gs2.png" alt="" style="width: 40%;"></div>   
        <p>with SE cor,i the standard errors(SE) of the mean values for the imposed naringenin concentrations 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9 and 1 mM. We did not take the concentration 0.0 mM naringenin or liquiritigenin and the corresponding response signal into consideration, because they dominate the Noise parameter value. </p>
    </div>
    <div class="section">
        <h1 id="O">Methods of PACE</h1>
        <p><strong style="font-size: 25px;">DP6 mutation rate assay</strong></p> 
        <p>MP6 was transformed into S2060 and plated on Chl-resistant 2×YT plates containing 100 mM D-Glucose. Monoclones were picked and shaken till saturation in 2×YT medium containing 25 mM D-Glu and Chl; the above bacterial solution was diluted 103-fold and grown to OD600 of 0.5-0.7. Samples were divided into two groups: one with 25 mM D-Glucose and the other with 25 mM arabinose, and incubated for 12h until saturation. Then,the bacterial solution was diluted by gradient and divided into two groups:one was applied to 2×YT plates containing 100 mM D-Glucose and Chl, the other was applied to 2×YT plates containing 100 mM D-Glucose and Rif. Culture them at 37°C for 18-24 h. The number of colonies of each culture on D-Glucose ± Rif plates was counted. Formula for calculating MP6 mutation rate (tbp: substituents per bp per generation): ubp = f/[R × ln(N//N0)], where f is the frequency of rifampin-resistant mutants (as compared with the glucose control), R is the number of unique sites yielding rifampin resistance (Liu et al. have identified only 21 sites across both rpoB clusters in their experiments),5 N is the final population size and N0 is the population size at which resistance is first observed (empirically determined to be B1.5107). </p>
        <p><strong style="font-size: 25px;">Transformation and storage of E. coli S2060 competent cells and strains prepared by chemical methods</strong></p>
        <p>Luciferase assay, phage propagation experiment and PACE were operated in the E. coli S2060 strain. <br>
            Competent cells are prepared by the method below. The saturated S2060 solution was added to a conical flask containing 2 x YT liquid medium (Solarbio) with appropriate antibiotics and diluted 100 times. The flask was then incubated in a shaker at 37℃ at 200-300 rpm/min until the OD600 of the bacterial solution was 0.4-0.6. We took and centrifuged 50 mL bacterial solution in the centrifugal tubes t 6500 rpm, 4℃ for 10 min. Discard the supernatant, and add 2 mL cold liquid LB and 2 mL cold 2×TSS (MgCl2, PEG 3350 and DMSO were added to LB medium at final concentrations of 20 mM, 10% and 5% respectively). Then, we froze the competent cells in the liquid nitrogen after mixing and dispensing them in the tubes and stored at -80℃.  <br>
            As for transformation, 100 μL competent cells defrost on the ice; add pre-cooled plasmid mixture (each kind of plasmids should < 100 ng; 3 kinds of plasmids can be transferred at most) and 100 μL 1×KCM (KCl, CaCl2 and MgCl2 were added to LB medium at final concentrations of 100 mM, 30 mM and 50 mM respectively). Put it on the ice for 10 min after being mixedthoroughly, heat shock at 42 ℃ for 75 s; put it on the ice for 2-3 min; add 500 μL SOC solution in it and put it in the biological shaker at 200-300 rpm/min for 1 h; centrifuged the bacterial solution at 8000 rpm for 2 min, discard the supernatant; plate the bacterial solution on the 2×YT solid culture (1.5% agar) with proper antibiotics after the bacterial solution being resuspended; culture at 37 ℃ for 16-18 h. A saturated bacterial solution containing the appropriate antibiotic is mixed in equal proportions with sterile glycerol and stored frozen at -80 ℃. <br> </p>
        <p><strong style="font-size: 25px;">Construction of selection phage</strong></p> 
        <p>Golden Gate cloning was used to assemble selection phage (SP). For Golden Gate assembly, Sap Ⅰ (New England BioLabs) acted as IIS restriction endonuclease along with T4 DNA ligase (New England BioLabs). Golden Gate cloning reaction mixture was prepared as followes: 3-5 ng/kb/µL donor plasmids and 1-3 ng/kb/µL receptor (pBT137-SplitC and pBT137-SplitD). And the mixtures experienced thermal cycles (37 ℃ for 5 min,16 ℃ for 5 min, 40 cycles, 60 ℃ 5 min, 4 ℃). After this process, assemblies were transferred into S2208 competent cells.</p>
        <p><strong style="font-size: 25px;">Activity-independent phage plaque assay</strong></p> 
        <p>Saturated S2208 bacterial broth was diluted with 2×YT liquid medium containing Amp and shaken at 37°C to OD600 : 0.6-0.9. Phage was diluted 3 times with sterilized water in a gradient of 102 . Transfer 10 μL of each different concentration of phage to a centrifuge tube. Add 150 μl of diluted S2208, then 20 μL of X-Gal (dissolve X-Gal in DMF to a final concentration of 2% (wt/vol)) solution, leave for 5 min, add 1mL of hot 2×YT top agar, mix well and pour the above mixture onto 2×YT solid agar tetrads at 37 °C for 5-8 h. Calculate the phage titer: Phage titer number of phage spots in the corresponding quadrant × dilution × 100. </p>
        <p><strong style="font-size: 25px;">Isolation of colonies</strong></p> 
        <p>Gently touch the plaque with the tip of a P10 pipette. Place the pipette tip into 2-3 mL of DRM3, grow for 16-20 h in a biological shaker at 37℃. Centrifuge at 8000 g for 2 min. The supernatant was collected and filtered in a 1 ml syringe equipped with a PVDF filter (13 mm 0.22 μm) to obtain a phage stock solution. Its concentration was determined using an activity-independent phage plaque assay. POI fragments can also be amplified by PCR for sequencing. </p> <br>
        <p><strong style="font-size: 25px;">Activity-dependent plaque test</strong></p> 
        <p>S2060 saturated bacterial solution transformed with AP was diluted 1000 times into DRM3 medium containing appropriate antibiotics, and the culture was shaken at 37 ℃ until OD600 grow to 0.4-0.6. Cells were infected with phage at a starting titer of approximately 104 PFU/ml and incubated at 37 ℃ for 16-20 h on a biological shaker. Centrifuge at 8000 g for 3 min. The supernatant was collected and filtered in a 1ml syringe equipped with a PVDF filter (13 mm 0.22 μm) to obtain phage stock solution, which was stored at -4 ℃.</p>
        <p><strong style="font-size: 25px;">Purification of SP</strong></p> 
        <p>In the above transformation step, the above plasmids were replaced with a mixture product of assembled SP plasmids, heat shocked for 75 s, left on ice for 2-3 min, and then the cell mixture was directly diluted into 10ml of antibiotic-free 2×YT liquid medium at 37 ℃ for 16-18 h at 200-300 rpm/min on a biologicalshaker. 2 mL of bacterial solution was put into a centrifuge tube and centrifuged at 8000 g for 3 min. The supernatant was collected and filtered in a 1 ml syringe (Nanjing Dingbei Biotechnology co., LTD.) equipped with a PVDF filter (13 mm 0.22 μm) (Nanjing Dingbei Biotechnology co., LTD.) to obtain the phage stock solution and stored at -4℃. </p>
    </div>
    <div class="section">
        <h1 id="R">REFERENCE</h1>
        <p> 1.	B.P. Hubbard, A.H. Badran J.A. Zuris et al, Continuous directed evolution of DNA-binding proteins to improve TALEN specificity. Nat Methods 12, 939-42, (2015).
           <br> DOI: <a style="color: black;" href="http://doi.org/10.1038/nmeth.3515">http://doi.org/10.1038/nmeth.3515</a>  <br>
             2.	A.H. Badran, V.M. Guzov Q. Huai et al, Continuous evolution of Bacillus thuringiensis toxins overcomes insect resistance. Nature 533, 58-63, (2016).
           <br> DOI: <a style="color: black;" href="http://doi.org/10.1038/nature17938">http://doi.org/10.1038/nature17938</a> <br>
             3.	J.C. Carlson, A.H. Badran D.A. Guggiana-Nilo et al, Negative selection and stringency modulation in phage-assisted continuous evolution. Nat Chem Biol 10, 216-22, (2014).
           <br> DOI: <a style="color: black;" href="http://doi.org/10.1038/nchembio.1453">http://doi.org/10.1038/nchembio.1453 </a> <br>
             4.	G.M. Morris, R. Huey W. Lindstrom et al, AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility. J Comput Chem 30, 2785-91, (2009).
           <br> DOI:<a style="color: black;" href="http://doi.org/10.1002/jcc.21256">http://doi.org/10.1002/jcc.21256</a>  <br>
             5.	A.H. Badran D.R. Liu, Development of potent in vivo mutagenesis plasmids with broad mutational spectra. Nat Commun 6, 8425, (2015).
           <br> DOI:<a style="color: black;" href=" http://doi.org/10.1038/ncomms9425 "> http://doi.org/10.1038/ncomms9425 </a><br>
            </p>
            <div style="height: 100px;"></div>
    </div>
    </div>
   <div class="side">
    <ul class="sidelist">
        <li class="si"><a href="#b"><h1>MATERIALS</h1></a></li>
        <li class="si"><a href="#p"><h1>METHODS</h1></a></li>
        <li class="si"><a href="#O"><h1>METHODS OF PACE</h1></a></li>
        <li class="si"><a href="#R"><h1>REFERENCE</h1></a></li>
    </ul>
   </div>
</div>




    <!-- footer -->
    <section class="footer fp">
        <div class="grid-wrapper">
            <div class="footer1">
                <h1 class="private">Tech Bees</h1>
                <p class="bottom">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Est ab beatae itaque ipsam
                    corporis maxime
                    saepe, voluptatum sapiente veritatis repellat!</p>
                <div class="social-profiles">
                    <i class="fa fa-facebook"></i>
                    <i class="fa fa-twitter"></i>
                    <i class="fa fa-instagram"></i>
                    <i class="fa fa-linkedin"></i>
                </div>
            </div>
            <div class="col3 footer2 fleft">
                <h1>Contact Info</h1>
                <h4>Lorem ipsum dolor sit.</h4>
                <h4>Lorem ipsum dolor sit.</h4>
                <h4>412616516</h4>
                <h4>@kjfhsdakf</h4>
                <h4>website.com</h4>
            </div>
            <div class="col3 footer3 fmid">
                <h1>Contact Us</h1>
                <h4>Company</h4>
                <h4>Events</h4>
                <h4>Projects</h4>
                <h4>Awards</h4>
                <h4>Open sources</h4>
            </div>
            <div class="col3 footer4">
                <h1>Custom Software</h1>
                <h4>Company</h4>
                <h4>Events</h4>
                <h4>Projects</h4>
                <h4>Awards</h4>
                <h4>Open sources</h4>
            </div>
        </div>
        <hr>
        <div class="copyright">
            <h4>Copyright Hyc</h4>
            <h4>Teams & Members</h4>
            <h4>Privacy</h4>
        </div>
        <!-- scroll to top btn -->
        <i id="scrolltp" class="fa fa-arrow-up"></i>
    </section>

    <!-- 导航栏二级 -->
    <script>

        var nav = document.querySelectorAll('.second');
        for (var i = 0; i < nav.length; i++) {
            nav[i].addEventListener('mouseenter', function () {
                var first = this.querySelector('.first');
                var secondChild = first.children[1];
                var img = this.querySelector('.ims');

                secondChild.style.opacity = "0";
                secondChild.style.transition = "opacity 0.5s";
                secondChild.style.display = "block";

                setTimeout(function () {
                    secondChild.style.opacity = "1";
                }, 200);

                img.classList.add("fade-out");

                img.addEventListener("transitionend", function () {
                    img.src = "../img/icon2.png";
                    img.classList.remove("fade-out");
                }, { once: true });
            });

            nav[i].addEventListener('mouseleave', function () {
                var first = this.querySelector('.first');
                var secondChild = first.children[1];
                var img = this.querySelector('.ims');

                secondChild.style.opacity = "0";
                secondChild.style.transition = "opacity 0.3s";

                img.classList.add("fade-out");

                img.addEventListener("transitionend", function () {
                    secondChild.style.display = "none";
                    img.src = "../img/icon1.png";
                    img.classList.remove("fade-out");
                }, { once: true });
            });
        }


    </script>
    <!-- normal -->
    <script src="../javascript/p_lbt.js"></script>

    <!-- AOS -->
    <script src="https://unpkg.com/aos@next/dist/aos.js"></script>
    <script>
       
    </script>
    <!-- App . js 图片轮播 -->
    <script src="../javascript/p_app.js"></script>

    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"></script>
</body>

</html>