<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>

    <!-- font -->
    <!-- <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" /> -->
    <link rel="stylesheet" href="https://cdn.staticfile.org/font-awesome/4.7.0/css/font-awesome.css">

    <!-- AOS -->
    <!-- <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" /> -->

    <!-- css -->
    <link rel="stylesheet" href="../css/p_project_css.css">

    <!-- bootstrap -->
    <link rel="stylesheet" href="../css/p_main.css">
    <style>
        table,
td,
th {
  text-align: center;
  border: 1px solid #000;
  border-collapse: collapse;
}       
    </style>
    
</head>

<body>
    <div id="toopp"></div>
    <div class="header">
        <div>
            <nav class="navbar navbar-expand-md">
                <a href="" class="navbar-brand"><img src="../img/logo.png" alt="" class="logo"></a>
                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navmenu">
                    <span class="navbar-toggler-icon"></span>
                </button>
                <div class="collapse navbar-collapse" id="navmenu">
                    <ul class="nav-list navbar-nav ms-auto">
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="../index.html">
                                    <div class="nav-link">Home</div>
                                </a>
                            </li>
                        </div>
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="../index.html">
                                    <div class="nav-link">Project</div>
                                </a>
                                <ul>
                                    <li><a href="../pages/Description.html">
                                            <div><img src="../img/icon2.png" alt="">Description</div>
                                        </a></li>
                                    <li><a href="../pages/Design.html">
                                            <div><img src="../img/icon2.png" alt="">Design</div>
                                        </a></li>
                                    <li><a href="../pages/Results.html">
                                            <div><img src="../img/icon2.png" alt="">Results</div>
                                        </a></li>
                                    <li><a href="../pages/Supplement.html">
                                            <div><img src="../img/icon2.png" alt="">Supplement</div>
                                        </a></li>
                                    <li><a href="../pages/Protocol.html">
                                            <div><img src="../img/icon2.png" alt="">Protocol</div>
                                        </a></li>
                                </ul>
                            </li>
                        </div>
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="../pages/notebook.html">
                                    <div class="nav-link">Lab Notebook</div>
                                </a>
                            </li>
                        </div>
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="#">
                                    <div class="nav-link">Team</div>
                                </a>
                                <ul>
                                    <li><a href="../pages/TEAM页.html">
                                        <div><img src="../img/icon2.png" alt="">Team</div>
                                    </a></li>
                                    <li><a href="#">
                                        <div><img src="../img/icon2.png" alt="">Collaboration</div>
                                    </a></li>
                                </ul>
                            </li>
                        </div>
                    </ul>
                </div>
            </nav>
        </div>

    </div><div class="headPicture">
        <img src="../img/result.jpg" alt="" class="notebookImg" >
        <img src="../img/RESULT-8.png" alt="" class="notebookText">
    </div>
    <div class="benti">
   <div class="page">
    <div class="section">
        <h1 id="b">Construction of mutation libraries</h1>
        <p>Mutation libraries were established using two strategies: error-prone PCR (epPCR) and site-directed mutagenesis. Two rounds of epPCR targeting the fdeR gene with the StarMut Random Mutagenesis Kit, constructing a mutation library of 5.9×106 mutants. Optimization of the PCR ion environment via gradient experiments resulted in a stable epPCR system with an appropriate mutation rate, ranging from 5 ~ 22 mutation bases per kilobase (kb) (Table S5). <br>
            Meanwhile, site-directed mutation was performed on the fdeR gene through seamless cloning, creating five combinations of mutation sites. Based on the known amino acid sequence of FdeR,1 the three-dimensional (3D) structure of FdeR was predicted using AlphaFold22 (Figure S3, S4A, and B). AutoDock 4.2.63 was employed to explore potential liquiritigenin binding sites on FdeR (Figure S4C, D, and E). Following energy minimization principle, the optimal docking result was selected from 50 docking results, demonstrating hydrogen bond interactions between FdeR's Arg220, Thr244, and Leu268 and liquiritigenin (Figure 1A and B). Besides, the docking analysis of FdeR and promoter indicated liquiritigenin is unlikely to cause steric hindrance with DNA (Figure S5). The binding energy of this conformation was within an reasonable range, justifying its selection for site-directed mutation. Subsequently, a full mutation scanning on amino acids within 3 Å of liquiritigenin identified mutations L268W, H170Y, and H170W (Table 1).4 Containing five mutation combinations: L268W, H170Y, H170W, L268W + H170Y and L268W + H170W, the site-directed mutation library was transformed into E. coli BL21 for subsequent fluorescence screening experiments.
        </p>
        
        <p><strong>Table 1. </strong> Main characteristics of FdeR single mutation prediction</p>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody><tr>
                <th>Index</th>
                <th>Mutation</th>
                <th>Mutation Energy (kcal/mol)</th>
                <th>Effect</th>
            </tr>
            <tr>
                <td>1</td>
                <td>A:HIS170>TRP</td>
                <td>-0.83</td>
                <td>STABILIZING</td>
            </tr>
            <tr>
                <td>2</td>
                <td>A:LEU268>TYR</td>
                <td>-0.73</td>
                <td>STABILIZING</td>
            </tr>
            <tr>
                <td>3</td>
                <td>A:HIS170>TYR</td>
                <td>-0.57</td>
                <td>STABILIZING</td>
            </tr>
        </tbody></table>
        <div style="display: flex;justify-content: center;margin-top: 50px;"><img src="../img/figure 1.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 1.</strong> (A) Hydrogen Bond Contacts between FdeR and liquiritigenin. Amino acid residues on FdeR that form hydrogen bond contacts with liquiritigenin are labeled and represented using ball-and-stick models. Yellow dashed lines represent hydrogen bonds, and the numbers labeled above indicate the bond lengths. (B) Interactions between FdeR and liquiritigenin, including hydrogen bonds and various π bonds, as well as the bond lengths for each type of bond.</p>
    </div>
    <div class="section">
        <h1 id="p">Mutant screening and fluorescence assay</h1>
        <p>With a sorting rate of up to 70,000 events per second, FACS technology was applied to screening for mutants that specifically and efficiently identify liquiritigenin to while having no or only low responsiveness to naringenin (Figure 2 and Figure S6). </p>
        <div style="display: flex;justify-content: center;margin-top: 70px;"><img class="gs" src="../img/Figure 2.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 2. Flow chart of mutants screening.</strong>(1) Add liquiritigenin to library; (2) Sort cells on the top 5% within the positive gate; (3) Add naringenin to sorted cells; (4) Sort cells on the bottom 5% within negative gate; (5) Inculate culture into 96 deep-well plates; (6) Resuspend with 1× PBS and transfer cell resuspension to 96-well plates; (7) Measure fluorescence by a microplate reader.</p>
        <p>In the first round of epPCR, six parallel epPCR systems were established for positive screening. Supplemented with 0.5 mM liquiritigenin and naringenin separately, positive selections and negative selections were performed on the library (Figure 3A and B). Subsequently, 320 individual clones were used for 96-well microplate screening.</p>
        <div style="display: flex;justify-content: center;margin-top: 50px;"><img src="../img/Figure 3.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 3. Flow cytometry analysis partly and FACS screening of the epPCR-2 and -4 library.</strong> Rounds of FACS screening were performed alternating positive sorting (library culture in the presence of 0.5 mM in the first round of epPCR or 0.35 mM in the second round of epPCR liquiritigenin and sorting of the events in the top of the histogram) and negative sorting (library culture in the presence of 0.5 mM in the first round of epPCR or 1.0 mM naringenin in the second round of epPCR and sorting of the events in the bottom of the fluorescence histogram). Round 1 of the epPCR library: (A) Positive screening was analyzed in a FCS-A vs. FITC-A scatter plot and the GFP Neg. (37.7%) and GFP Pos. (7.29%) gates were set. An Events vs. FITC-A histogram plot was created and the Pos. Sort gate was delimitated to sort the 7.29% most fluorescent events. (B) Negative screening was analyzed in a FCS-A vs. FITC-A scatter plot and the GFP Neg. (56.5%) and GFP Pos. (0.39%) gates were set. An Events vs. FITC-A histogram plot was created and the Pos. Sort gate was delimitated to sort the 5.77% least fluorescent events. Rouns 2 of the epPCR library: (C) Positive screening was analyzed in a FCS-A vs. FITC-A scatter plot and the GFP Neg. (54.1%) and GFP Pos. (20.3%) gates were set. An Events vs. FITC-A histogram plot was created and the Pos. Sort gate was delimitated to sort the 5.96% most fluorescent events. (D) Negative screening was analyzed in a FCS-A vs. FITC-A scatter plot and the GFP Neg. (18.7%) and GFP Pos. (6.92%) gates were set. An Events vs. FITC-A histogram plot was created and the Pos. Sort gate was delimitated to sort the 6.91% least fluorescent events.</p>
        <p style="margin-top: 50px;">This yielded a mutant library with a 2-10-fold increase in liquiritigenin response intensity compared to the wild type (Figure 4A). Among them, Mut_49 stood out, displaying a remarkable 10-fold increase in peak fluorescence compared to the wild type. To further enhance the sensitivity and specificity of this biosensor in responding to liquiritigenin, we subjected four variants, namely Mut_39, Mut_49, Mut_58, and Mut_81, with favorable fluorescence characteristics to the next round of epPCR (Figure 3C and D). Finally, Mut_49_70 with a fluorescence intensity nearly 100 times higher than the wild type was isolated (Figure 4B), indicating a significant enhancement in this biosensor in responding to liquiritigenin.</p>
        <div style="display: flex;justify-content: center;margin-top: 50px;"><img src="../img/Figure 4.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 4. Comparison of the fluorescence intensities of mutants and the wild-type to liquiritigenin.</strong> (A) Mutants obtained from the first round epPCR. (B) Mutants obtained from the second round epPCR. Mutants and wild-type were induced with 1 mM liquiritigenin for 12 hours. The green bars represent the fluorescence intensities of different mutants, while the blue bars represent the fluorescence intensity of the wild-type.</p>
        <p>In site-directed mutation experiments, we performed a 96-deep well microplate screening with 1.0 mM liquiritigenin or 1.0 mM naringenin. Regrettably, the mutants exhibit no significant increase in response intensity to liquiritigenin or naringenin compared to the wild-type FdeR (Figure 5). Consequently, a dose-response analysis for these mutants was not conducted, necessitating a reevaluation of our bioinformatics-based predictions.</p>
        <p>R.	Wassem et al.<sup>5</sup> demonstrated that FdeR recognizes and binds to the PfdeA promoter, inhibiting RNA polymerase access. Upon binding flavonoids, like naringenin, FdeR undergoes conformational changes altering its DNA-binding site, enabling RNA polymerase binding and transcription initiation. Initial molecular docking focused on the individual FdeR monomer interaction with liquiritigenin, neglecting dynamic changes in FdeR binding to the PfdeA promoter and its oligomeric state in vivo. Future research will readdress this aspect through molecular docking to identify novel mutation sites.<sup>5</sup></p>
        <div  style="display: flex;justify-content: center;margin-top: 50px;"><img class="gs" src="../img/figure 5.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 5. Dose responses to liquiritigenin and naringenin of different mutantgensis constructs. </strong>Wild-type (WT) and five combinations of mutantgensis L268W, H170Y, H170W, H170W + L268W, H170Y + L268W were cultivated for 12 h in the presence of increasing concentrations of liquiritigenin (blue) and naringenin (red) in 500 µM. Error bars are based on the standard deviation of a minimum of biological triplicate.</p>
    </div>
    <div class="section">
        <h1 id="O">Performance testing of mutant PfdeAR-FdeR sensors</h1>
        <p>Based on the 96-well microplate fluorescence screening, 8 mutants with high fluorescence intensity responses to liquiritigenin were selected for gradient experiments, varying liquiritigenin concentration and induction time, to characterize their performance as biosensor, including sensitivity, detection range, response time, and specificity, modeled using Hill functions to analyze dose impact.5 We defined operational range as the ligand concentration range and response intensity as fluorescence intensity divided by OD600. We also introduced a Noise parameter to quantify average relative error in fluorescence response across concentrations, a vital aspect in biosensor development and application.</p>
        <p>Compared to the initial state, all 8 mutants showed substantial 10-100-fold enhancements in fluorescence response intensity (Figure 4B), exhibiting a positive correlation within the 0-1 mM liquiritigenin concentration range, with most reaching a plateau at 0.5 mM liquiritigenin (Figure 6, Table S4). Notably, Mut_49_70 and Mut_49_39 exhibited more higher fluorescence intensity and a broader detection range. Even at 1 mM liquiritigenin, their response curves continued to show an upward trend, while Noise parameters both remained around 20%. Next, the response time capabilities of the eight mutants were assessed (Figure 7). While the wild-type FdeR displayed an S-shaped time response curve, with fluorescence intensity increasing soomthly within 6 hours and peaking at 10 hours, Mut_49_70 exhibited a significantly longer response time. It showed rapid early-stage intensity rise within 2 hours, reaching its peak at approximately 12 hours, indicating Mut_49_70 can respond quickly to liquiritigenin in the environment.</p>
        <div style="display: flex;justify-content: center;margin-top: 50px;"><img src="../img/Figure 6. 原图Results-liquiritigenin response curve  of our mutants.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 6. The dose-response curves for liquiritigenin of the wild type and the mutants are as follows:</strong>The blue curve represents the dose-response curve for the wild type. Various green curves represent the dose-response curves for the 8 mutants.</p>
        <div style="display: flex;justify-content: center;margin-top: 50px;"><img src="../img/Figure 7.原图 time response curve.jpg" alt="" style="width: 70%;"></div>
        <p><strong>Figure 7. The response time curves for liquiritigenin of the wild type and Mut_49_70 are as follows:</strong>The blue curve represents the response time curve for the wild type. The green curve represents the response time curve for the mutant Mut_49_70. </p>
    <p>To evaluate specificity, we conducted dose-response experiments for naringenin in the 8 mutants (Figure 8 and Figure S8). These mutants exhibited strong specificity, with most displaying over a 5-fold difference in response intensity to liquiritigenin compared to naringenin. Notably, Mut_49_70 showed the highest fluorescence intensity among the mutants, exceeding 1.7×107 a.u.. However, its response intensity to naringenin also ranked the highest, reaching approximately 2.1×106 a.u. (still lower than wild-type FdeR). And Mut_49_39 exhibited impressive fluorescence response intensity, reaching up to 1.2×107 a.u., with significantly stronger specificity for liquiritigenin than Mut_49_70. Mut_49_39 displayed a broader dynamic range, maintaining an upward trend in fluorescence intensity even at a 1.0 mM liquiritigenin concentration, which is often considered as an ideal characteristic for a biosensor.</p>
    <div style="display: flex;justify-content: center;margin-top: 50px;"><img src="../img/Figure 8改.png" alt="" style="width: 70%;"></div>
    <p><strong>Figure 8. Fluorescence intensities of different mutants to liquiritigenin and naringenin.</strong>(A) Fluorescence intensities of 8 mutants at 1 mM liquiritigenin and naringenin, where green bars represent the liquiritigenin, and orange bars represent the naringenin. (B) The dose-response curves for the Mut_49_39 mutant to liquiritigenin and naringenin. (C) The dose-response curves of the Mut_49_70 mutant to liquiritigenin and naringenin. The green curve represents liquiritigenin, and the orange curve represents naringenin in the dose-response curves for both ligands. </p>
    <p>Additionally, We identified mutations, including D71E, S140P, M191T, and S297C, enriched in most mutants. Remarkably, elongation mutations were observed in Mut_33, Mut_48, Mut_49, and Mut_64, enhancing their responsiveness and specificity to liquiritigenin (Figure 9A and 4). Compared to Mut_49, F106L and V134L mutations enriched in Mut_49_75, Mut_49_77, Mut_49_78, Mut_49_18, and Mut_49_36, while E163V and I308S mutations were enriched in Mut_49_40 and Mut_49_41 (Figure 9B). And Mut_49_39 and Mut_49_70 showed no additional mutations, but an mutation (G to A) was detected between the promoter and RBS, potentially influencing biosensor performance at the transcriptional or translational level, resulting in both mutants constructing the best-performing biosensors. These sequencing results serve as the foundation for subsequent molecular dynamics simulations.</p>
    <p>In summary, among the 8 mutants studied, Mut_49_39 and Mut_49_70 demonstrated a broader response range (exceeding 1 mM), faster response kinetics (within 2 hours), and a relatively normal Noise parameter (about 20%). Consequently, Mut_49_39 and Mut_49_70 hold substantial potential for constructing highly efficient and specific liquiritigenin biosensors.</p>
    <div style="display: flex;justify-content: center;margin-top: 50px;"><img src="../img/Figure 9改.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 9. The amino acid mutation heatmap of the mutants.</strong>(A) After the first round of error-prone PCR, six mutants were identified. Among them, four mutants exhibited enrichment in mutations at S140P, M191T, and S297C, and leading to extended mutations spanning twelve amino acids. (B) The mutants obtained after the second round of error-prone PCR, using Mut_49 as a template, were sequenced and analyzed. </p>
</div>
    <div class="section">
        <h1 id="R">Docking and Molecular Dynamic Simulations</h1>
        <p>After predicting the three-dimensional structure of Mut_49, this mutant was docked with liquiritigenin, analying changes in intermolecular forces and binding pocket hydrophobicity(Figure 10). The results indicated that Mut_49 revealed higher hydrophobicity in the binding region compared to the wild-type, suggesting enhanced affinity and specificity for liquiritigenin. Unfortunately, upon analyzing its mutation energy, it was found that all mutations were neutral, contradicting the experimental results.</p>
        <div style="display: flex;justify-content: center;margin-top: 50px;"><img src="../img/figure 10改.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 10. </strong>(A) Hydrogen bond interactions between mutant 49 and liquiritigenin. Liquiritigenin and the amino acid residues of mutant 49 that have hydrogen bond interactions are labeled and shown in ball-and-stick model. Yellow dashed lines represent hydrogen bonds, and the numbers on them indicate the hydrogen bond lengths. (B) The intermolecular forces between mutant 49 and liquiritigenin, such as hydrogen bonds and various π bonds, and the bond lengths of each type of bond.</p>
        
        <p><strong>Table 2. </strong> Main characteristics of mutant 49 mutation prediction</p>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody><tr>
                <th>Index</th>
                <th>Mutation</th>
                <th>Mutation Energy (kcal/mol)</th>
                <th>Effect</th>
            </tr>
            <tr>
                <td>1</td>
                <td>A: VAL134>LEU</td>
                <td>0.05</td>
                <td>NEUTRAL</td>
            </tr>
            <tr>
                <td>2</td>
                <td>A: PHE106>LEU</td>
                <td>0.07</td>
                <td>NEUTRAL</td>
            </tr>
            <tr>
                <td>3</td>
                <td>A: PHE106>LEU</td>
                <td>0.07</td>
                <td>NEUTRAL</td>
            </tr>
            <tr>
                <td></td>
                <td>A: PHE13>LEU</td>
                <td></td>
                <td>NEUTRAL</td>
            </tr>
            <tr>
                <td>4</td>
                <td>A: MET60>LEU</td>
                <td>0.00</td>
                <td>NEUTRAL</td>
            </tr>
            <tr>
                <td>5</td>
                <td>A: HIS246>GLU</td>
                <td>0.41</td>
                <td>NEUTRAL</td>
            </tr>
        </tbody></table>
        <p>We then carried out molecular dynamics simulations on the complexes of FdeR with liquiritigenin and mutant 49 with liquiritigenin, respectively.6 The structural changes and the binding between WT and liquiritigenin were investigated by viewing the system potential energy, Rg, RMSD, and interaction energy (Figure 11). MMPBSA analysis was carried out to correlate the MD data with peptide characteristics (Table 3). The interaction energy of the WT was about 15% lower than that of MU, demonstrating that the binding affinity between the WT and the receptor was much higher than the mutant type.</p>
        <div style="display: flex;justify-content: center;margin-top: 50px;"><img src="../img/Figure 11改.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 11. Molecular dynamics trajectory analysis of WT and mutant 49.</strong>WT stands for FdeR and MU stands for mutant 49. (A) The RMSD curve of the protein during the protein-ligand complex simulation process (the complex reached equilibrium after 20 ns, indicating that the whole simulation process was stable and reliable); (B) The Rg curve of the protein during the protein-ligand complex simulation process (the radius of gyration (Rg) is used to prove the compactness of the protein structure during the simulation process, it is the distance between the centroid of all atoms and their ends at a specific time interval. During the whole complex MD simulation process, the Rg value fluctuation of MU was larger than that of WT, indicating that WT might bind more tightly with the small molecule); (C) The RMSF curve of the protein during the protein-ligand complex simulation process (the fluctuation of each atom relative to its average position is calculated, which characterizes the average of the structural changes over time, and gives a characterization of the flexibility of each region of the protein. The RMSF value of the protein in the MU-ligand complex group was larger than that of the protein in the WT-ligand complex group in the 0-75 range, indicating that the flexibility of MU was greater than that of WT).</p>
        <p><strong>Table 3. </strong> Comparison of statistics between WT and MU</p>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody><tr>
                <th>Energy</th>
                <th>WT</th>
                <th>MU</th>
            </tr>
            <tr>
                <td>Van der Waals Energy (KJ/mol)</td>
                <td>-156.689</td>
                <td>-131.048</td>
            </tr>
            <tr>
                <td>Electrostatic Energy (kJ/mol)</td>
                <td>-29.978</td>
                <td>-7.192</td>
            </tr>
            <tr>
                <td>Polar solvation Energy (KJ/mol)</td>
                <td>101.397</td>
                <td>67.802</td>
            </tr>
            <tr>
                <td>Nonpolar solvation Energy (KJ/mol)</td>
                <td>-16.180</td>
                <td>-17.227</td>
            </tr>
            <tr>
                <td>Total Binding Energy (KJ/mol)</td>
                <td>-101.451</td>
                <td>-87.666</td>
            </tr>
            <tr>
                <td>T∆S(KJ/mol)</td>
                <td>4.444</td>
                <td>10.645</td>
            </tr>
            <tr>
                <td>Total Binding Free Energy (KJ/mol)</td>
                <td>-97.007</td>
                <td>-77.021</td>
            </tr>
        </tbody></table>
    </div>
    <div class="section">
        <h1 id="R2">Riboswitch directed evolution via phage-assisted continuous evolution (PACE)</h1>
        <p>Currently, we have successfully constructed the plasmids (SP, AP and CP) required for PACE (Figure 12, Figure S13,S14).</p>
        <div style="display: flex;justify-content: center;margin-top: 20px;"><img src="../img/Figure 12改.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 12. Activity-independent phage plaque assay.</strong>(A) SP-Mid2 phage and (B) SP-High2 phage; from the upper right quadrant, upper left quadrant, lower right quadrant, and lower left quadrant, the phage concentrations are 100, 10-2, 10-4, 10-6 in order, and the number of phages in appropriate in quadrants can be used to quantify the titer.</p>
        <p>We also successfully built the PACE device, tuned the operational parameters of syringe pumps and peristaltic pumps, as well as the length of needles in the chemostat and lagoons(Figure 13). This ensured the host bacteria maintained a stable cell density and allowed M13 phage propagate normally in the lagoons. The mutation rate μbp of DP6 has been measured as 1.007×10-5, and the PACE system has been initiated. We will isolate promising individuals from this pool and further test their performance as biosensors.</p>
        <div style="display: flex;justify-content: center;margin-top: 20px;"><img src="../img/Figure 13改.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure 13. </strong>Schematic of the PACE apparatus. Coupled to each other, allowing the phage to continuously cross-seed between the two lagoons. Chemostat. Provide constant cell density for lagoons for M13 bacteriophage infection. Lagoons, Phage infect and propagate. Inducers. Initiate the expression of mutagenic genes to mutate at the genome-wide level of E. coli. (B) Photo of our PACE apparatus. The picture on the left shows syringe pumps(0.3 ml/h) and peristaltic pumps(0.258 ml/min). The picture on the right shows DRM carboy, the chemostat and lagoons.</p>
    </div>
    <div class="section">
        <h1 id="R3">REFERENCE</h1>
        <p>1.J. Vandierendonck, Y. Girardin P. De Bruyn et al, A Multi-Layer-Controlled Strategy for Cloning and Expression of Toxin Genes in  Escherichia coli. Toxins (Basel) 15, (2023). <br>
            DOI: <a style="color: #000;" href="http://doi.org/10.3390/toxins15080508 ">http://doi.org/10.3390/toxins15080508 </a> <br>
            2.	 T.P. Huang, Z.J. Heins S.M. Miller et al, High-throughput continuous evolution of compact Cas9 variants targeting  single-nucleotide-pyrimidine PAMs. Nat Biotechnol 41, 96-107, (2023). <br>
            DOI: <a style="color: #000;" href="http://doi.org/10.1038/s41587-022-01410-2">http://doi.org/10.1038/s41587-022-01410-2</a>  <br>
            3.	G.M. Morris, R. Huey W. Lindstrom et al, AutoDock4 and AutoDockTools4: Automated docking with selective receptor  flexibility. J Comput Chem 30, 2785-91, (2009). <br>
            DOI: <a style="color: #000;" href="http://doi.org/10.1002/jcc.21256 ">http://doi.org/10.1002/jcc.21256 </a> <br>
            4.	BIOVIA, Dassault Systèmes, Discovery Studio Visualizer, v21.1.0.20298, San Diego: Dassault Systèmes, 2020.
            5.	R. Wassem, A.M. Marin A. Daddaoua et al, A NodD-like protein activates transcription of genes involved with naringenin  degradation in a flavonoid-dependent manner in Herbaspirillum seropedicae. Environ Microbiol 19, 1030-1040, (2017). <br>
            DOI: <a style="color: #000;" href="http://doi.org/10.1111/1462-2920.13604 ">http://doi.org/10.1111/1462-2920.13604 </a><br>
            6.	Y. Jiang, J. Gu J. Lei et al, Molecular Dynamics and Bioactivity of a Novel Mutated Human Parathyroid Hormone. Tropical Journal of Pharmaceutical Research 13, 511-518, (2014). <br>
            DOI: <a style="color: #000;" href="http://doi.org/10.4314/tjpr.v13i4.4">http://doi.org/10.4314/tjpr.v13i4.4</a> <br>
            </p>
    </div>
    </div>
   <div class="side s3" style="height: 560px;">
    <ul class="sidelist">
        <li class="si"><a href="#b"><h1>Construction of mutation libraries</h1></a></li>
        <li class="si"><a href="#p"><h1>Mutant screening and fluorescence assay</h1></a></li>
        <li class="si"><a href="#O"><h1>Performance testing of mutant PfdeAR-FdeR sensors</h1></a></li>
        <li class="si"><a href="#R"><h1>Docking and Molecular Dynamic Simulations</h1></a></li>
        <li class="si"><a href="#R2"><h1>Riboswitch directed evolution via phage-assisted continuous evolution (PACE)</h1></a></li>
        <li class="si"><a href="#R3"><h1>REFERENCE</h1></a></li>
    </ul>
   </div>
</div>




    <!-- footer -->
    <section class="footer fr">
        <div class="grid-wrapper">
            <div class="footer1">
                <h1 class="private">Tech Bees</h1>
                <p class="bottom">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Est ab beatae itaque ipsam
                    corporis maxime
                    saepe, voluptatum sapiente veritatis repellat!</p>
                <div class="social-profiles">
                    <i class="fa fa-facebook"></i>
                    <i class="fa fa-twitter"></i>
                    <i class="fa fa-instagram"></i>
                    <i class="fa fa-linkedin"></i>
                </div>
            </div>
            <div class="col3 footer2 fleft">
                <h1>Contact Info</h1>
                <h4>Lorem ipsum dolor sit.</h4>
                <h4>Lorem ipsum dolor sit.</h4>
                <h4>412616516</h4>
                <h4>@kjfhsdakf</h4>
                <h4>website.com</h4>
            </div>
            <div class="col3 footer3 fmid">
                <h1>Contact Us</h1>
                <h4>Company</h4>
                <h4>Events</h4>
                <h4>Projects</h4>
                <h4>Awards</h4>
                <h4>Open sources</h4>
            </div>
            <div class="col3 footer4">
                <h1>Custom Software</h1>
                <h4>Company</h4>
                <h4>Events</h4>
                <h4>Projects</h4>
                <h4>Awards</h4>
                <h4>Open sources</h4>
            </div>
        </div>
        <hr>
        <div class="copyright">
            <h4>Copyright Hyc</h4>
            <h4>Teams & Members</h4>
            <h4>Privacy</h4>
        </div>
        <!-- scroll to top btn -->
        <i id="scrolltp" class="fa fa-arrow-up"></i>
    </section>

    <!-- 导航栏二级 -->
    <script>

        var nav = document.querySelectorAll('.second');
        for (var i = 0; i < nav.length; i++) {
            nav[i].addEventListener('mouseenter', function () {
                var first = this.querySelector('.first');
                var secondChild = first.children[1];
                var img = this.querySelector('.ims');

                secondChild.style.opacity = "0";
                secondChild.style.transition = "opacity 0.5s";
                secondChild.style.display = "block";

                setTimeout(function () {
                    secondChild.style.opacity = "1";
                }, 200);

                img.classList.add("fade-out");

                img.addEventListener("transitionend", function () {
                    img.src = "../img/icon2.png";
                    img.classList.remove("fade-out");
                }, { once: true });
            });

            nav[i].addEventListener('mouseleave', function () {
                var first = this.querySelector('.first');
                var secondChild = first.children[1];
                var img = this.querySelector('.ims');

                secondChild.style.opacity = "0";
                secondChild.style.transition = "opacity 0.3s";

                img.classList.add("fade-out");

                img.addEventListener("transitionend", function () {
                    secondChild.style.display = "none";
                    img.src = "../img/icon1.png";
                    img.classList.remove("fade-out");
                }, { once: true });
            });
        }


    </script>
    <!-- normal -->
    <script src="../javascript/p_lbt.js"></script>

    <!-- AOS -->
    <script src="https://unpkg.com/aos@next/dist/aos.js"></script>
    <script>
       
    </script>
    <!-- App . js 图片轮播 -->
    <script src="../javascript/p_app.js"></script>

    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"></script>
</body>

</html>