<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Supplement</title>

    <!-- font -->
    <!-- <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" /> -->
    <link rel="stylesheet" href="https://cdn.staticfile.org/font-awesome/4.7.0/css/font-awesome.css">

    <!-- AOS -->
    <!-- <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" /> -->

    <!-- css -->
    <link rel="stylesheet" href="../css/p_project_css.css">

    <!-- bootstrap -->
    <link rel="stylesheet" href="../css/p_main.css">
    <style>
        table,
td,
th {
  text-align: center;
  border: 1px solid #000;
  border-collapse: collapse;
}       
    </style>
    
</head>

<body>
    <div id="loader">
        <img src="../img/loader.gif" alt="Loading...">
    </div>
    <div id="toopp"></div>
    <div class="header">
        <div>
            <nav class="navbar navbar-expand-md">
                <a href="" class="navbar-brand"><img src="../img/logo.png" alt="" class="logo"></a>
                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navmenu">
                    <span class="navbar-toggler-icon"></span>
                </button>
                <div class="collapse navbar-collapse" id="navmenu">
                    <ul class="nav-list navbar-nav ms-auto">
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="../index.html">
                                    <div class="nav-link">Home</div>
                                </a>
                            </li>
                        </div>
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="../index.html">
                                    <div class="nav-link">Project</div>
                                </a>
                                <ul>
                                    <li><a href="../pages/Description.html">
                                            <div><img src="../img/icon2.png" alt="">Description</div>
                                        </a></li>
                                    <li><a href="../pages/Design.html">
                                            <div><img src="../img/icon2.png" alt="">Design</div>
                                        </a></li>
                                    <li><a href="../pages/Results.html">
                                            <div><img src="../img/icon2.png" alt="">Results</div>
                                        </a></li>
                                    <li><a href="../pages/Supplement.html">
                                            <div><img src="../img/icon2.png" alt="">Supplement</div>
                                        </a></li>
                                    <li><a href="../pages/Protocol.html">
                                            <div><img src="../img/icon2.png" alt="">Protocol</div>
                                        </a></li>
                                </ul>
                            </li>
                        </div>
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="../pages/notebook.html">
                                    <div class="nav-link">Lab Notebook</div>
                                </a>
                            </li>
                        </div>
                        <div class="second">
                            <img class="ims" src="../img/icon1.png" alt="">
                            <li class="nav-item cool-link first"><a href="#">
                                    <div class="nav-link">Team</div>
                                </a>
                                <ul>
                                    <li><a href="../pages/TEAM页.html">
                                        <div><img src="../img/icon2.png" alt="">Team</div>
                                    </a></li>
                                    <li><a href="#">
                                        <div><img src="../img/icon2.png" alt="">Collaboration</div>
                                    </a></li>
                                </ul>
                            </li>
                        </div>
                    </ul>
                </div>
            </nav>
        </div>

    </div>
    <div class="headPicture">
        <img src="../img/supplement.jpg" alt="" class="notebookImg">
        <img src="../img/SUPPLEMENT-8.png" alt="" class="notebookText">
    </div>
    <div class="benti">
   <div class="page">
    <div class="section">
        <h1 id="b">FIGURE</h1>
       
            <div style="display: flex;justify-content: center;margin-top: 20px;"><img src="../img/s1.png" alt="" style="width: 70%;"></div>
            <p> <strong>Figure S1. Optimization of the measurement signal-to-noise ratio of eGFP wavelengths.</strong>  An optimal combination of excitation and emission wavelengths (Ex = 482 nm / Em = 524) was obtained and applied for subsequent detection by the spectral optimization wizard in SoftMax Pro software.</p>
            
            <div style="display: flex;justify-content: center;"><img src="../img/s2.png" alt="" style="width: 70%;"></div>
            <p><strong>Figure S2. The growth curves of Escherichia coli. (A)</strong>Under the different concentration of naringenin. (B) Under the different concentration of liquiritigenin. Concentrations of both less than 1 mM barely affect bacterial growth. But the concentration of both at 2.0 mM and 3.0mM began to affect the normal growth of bacteria.</p>
            <div style="display: flex;justify-content: center;"><img src="../img/s3.png" alt="" style="width: 70%;"></div>
            <p><strong>Figure S3. </strong> (A) 3D structure of naringenin. (B) 3D structure of and liquiritigenin. </p>
        <div style="display: flex;justify-content: center;"><img src="../img/s4.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure S4. Structure prediction and analysis of FdeR.</strong>(A) 3D structure of FdeR predicted by AlphaFold2. (B) pLDDT is a scoring system that measures the confidence of a predicted protein structure. It stands for per-residue lDDT, where lDDT is a metric that compares the local distances between two structures. pLDDT assigns a score between 0 and 100 to each residue in the predicted structure, based on how well it matches the reference structure. A higher score means a higher confidence in the prediction. The average pLDDT score across all residues indicates the overall confidence for the whole protein chain. (C) The best conformation of FdeR and liquiritigenin. (D) Hydrophobicity of binding area. (E) Binding energy and ranking of FdeR and liquiritigenin.</p>
       
        <div style="display: flex;justify-content: center;"><img src="../img/s5.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure S5. FdeR-promoter docking.</strong>(A) The structure of the P22 constitutive promoter modeled by Discovery Studio. (B) Docking results show relative positions between them   (C) Docking results including ranking and energy. (D) Relationships between frequency and scoring energy.</p>
        <div style="display: flex;justify-content: center;"><img src="../img/s6.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure S6. Flow cytometry analysis and gating strategy for E. coli BL21 control cells. </strong> (A) Cultures of E. coli BL21 transformed with plasmids containing eGFP reporter (Positive control)  was analyzed in a FCS-A vs. FITC-A scatter plot and a postive fluorescence gate (GFP Pos.) was delimitated (94.2%). (B) The GFP Pos. gate(FITC-A > 3.6×103) was observed in an Events vs. FITC-A histogram plot for better visualization. (C) Cultures of E. coli BL21 with empty pQE-80L plasmids (Negative control) was analyzed in a FCS-A vs. FITC-A scatter plot and a negative fluorescence gate (GFP Neg.) was delimitated (99.7%). (D) The GFP Neg. gate(FITC-A < 3 ×103) was observed in an Events vs. FITC-A histogram plot for better visualization.</p>
        
        <div style="display: flex;justify-content: center;"><img src="../img/s7.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure S7. Dilution spread of cell suspension obtained from FACS.</strong>Samples were diluted 0, 10, 100-fold and spread on plates to isolate single colonies, each representing a mutant, for subsequent 96-well plates screening.</p>
        
        <div style="display: flex;justify-content: center;"><img src="../img/s8.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure S8. Response curves of mutants to different concentrations of liquiritigenin and naringenin. </strong> (A) Mut_49_70. (B) Mut_49_39. (C) Mut_49_36. (D) Mut_49_18. (E) Mut_49_92. The response curve for naringenin is not applicable. (F) Mut_49_49. (G) Mut_49_66. The response curve for naringenin is not applicable. (H) Mut_49_11.</p>
       
        <div style="display: flex;justify-content: center;"><img src="../img/s9.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure S9. Structure prediction and analysis of Mut_49.</strong>(A) 3D structure of mutant49 predicted by AlphaFold2. (B) pLDDT scoring system. (C) The best conformation of Mut_49 and liquiritigenin. (D) Hydrophobicity of binding area. (E) Binding energy and ranking of mutant 49 and liquiritigenin. </p>
        
        <div style="display: flex;justify-content: center;"><img src="../img/s10.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure S10. Mut_49-promoter docking.</strong>(A)  Docking results show relative positions between them. (B) Docking results including ranking and energy. (C) Relationships between frequency and scoring energy.</p>
        
        <div style="display: flex;justify-content: center;"><img src="../img/s11.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure S11. Molecular dynamics trajectory analysis of WT and mutant 49 WT stands for FdeR, MU stands for Mut_49.</strong>(A) The number of hydrogen bonds during the protein-ligand complex simulation process (The average number of hydrogen bonds for WT and MU protein with the ligand are 1.23 and 0.34 respectively). (B) The SASA curve of the protein during the protein-ligand complex simulation process (The solvent accessible surface area (SASA) is calculated by the van der Waals interaction between the solute and the solvent molecules. The SASA of the protein decreases as the protein density increases, so the change of SASA can predict the change of protein structure. In the figure, the SASA values of the protein in all the complex simulation processes show a downward trend, indicating that the protein density structure increases, and the density change of MU protein is greater than that of WT protein).</p>
        
        <div style="display: flex;justify-content: center; height: 550px;width: 500px;margin: 0 auto;"><img src="../img/s12.png" alt="" style="width: 100%; object-fit: cover;object-position: left top;"></div>
        <p><strong>Figure S12. The map of plasmids used for directed evolution of allosteric transcription factors. (A) pQE-80L-p22-fdeR. </strong>This plasmid contains the fdeR biosensor sequence. (B) pQE-80L-eGFP-Pos. The plasmid contains Spacer, PfdeAR and eGFP and is used as positive control in FACS. (C) pQE-80L-Neg. The plasmid without fdeR biosensor sequence is used as negative control in FACS. </p>
        
        <div style="display: flex;justify-content: center;"><img src="../img/s13.png" alt="" style="width: 70%;"></div>
        <p><strong>Figure S13. The Selection Phage plasmids map for PACE. (A) SP-Mid2. </strong>The SP plasmid is based on the M13 wild-type phage with the gIII removed, plus the riboswitch Mid2 to be mutated and T7 RNA polymerase. (B) SP-High2. The SP plasmid is based on the M13 wild-type phage with the gIII removed, plus the riboswitch Mid2 to be mutated and T7 RNA polymerase.</p>
        
        <div style="display: flex;justify-content: center; height: 450px;width: 500px;margin: 0 auto; "><img src="../img/s14.png" alt="" style="width: 100%; object-fit: cover;object-position: left top;"></div>
        <p><strong>Figure S14. The Accessory Plasmids map for PACE. (A) Accessory Plasmid 1 (AP1).</strong>The plasmid containing M13 gⅢ and an eGFP downstream controlled by T7 promoter, is used in the positive selection in PACE. (B) Accesspry Plasmid 2 (AP2). The plasmid containing M13 gⅢ-neg and mRFP1 downstream controlled by T7 promoter, is used in the negative selection in PACE. (C) AP-P-Mid2-T7RNAP. The plasmid containing the riboswitch Mid2 and T7RNA polymerase sequence, is used in the Selection Phage (SP) assembly. (D) AP-P-Mid2-T7RNAP. The plasmid containing the riboswitch High2 and T7RNA polymerase sequence, is used in the Selection Phage (SP) assembly. </p>
    </div>
    <div class="section s2">
        <h1 id="p">TABLE</h1>
        <p><strong>Table S1. Escherichia coli Strains which were used in this study.</strong></p><br>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;"   >
            <tbody><tr>
                <th>Names</th>
                <th>Relevant characteristics</th>
                <th>Source</th>
            </tr>

            <tr>
                <td>DH5α</td>
                <td>deoR endA1 gyrA96 hsdR17 (rk-mk+)recA1 relA1 supE44 thi-1 Δ(lacZYA-argF)U169 Φ80lacZ ΔM15F - λ -</td>
                <td>Vazyme</td>
            </tr>
            <tr>
                <td>BL21(DE3）</td>
                <td>F-ompThsdS(rB-mB-) galdcm (DE3)</td>
                <td>Vazyme</td>
            </tr>
            <tr>
                <td>S2060</td>
                <td>F’ proA+B+ Δ(lacIZY) zzf::Tn10 lacIQ1 PN25-tetR luxCDE Ppsp(AR2) lacZ luxR Plux groESL / endA1 recA1 galE15 galK16 nupG rpsL
                    ΔlacIZYA araD139 Δ(ara,leu)7697 mcrA Δ(mrr-hsdRMS-mcrBC) proBA::pir116 araE201 ΔrpoZ Δflu ΔcsgABCDEFG ΔpgaC λ–
                    </td>
                <td>addgene</td>
            </tr>
            <tr>
                <td>S2208</td>
                <td>F’ proA+B+ Δ(lacIZY) zzf::Tn10 lacIQ1 PN25-tetR luxCDE Ppsp(AR2) lacZ luxR Plux groESL / endA1 recA1 galE15 galK16 nupG rpsL
                    ΔlacIZYA araD139 Δ(ara,leu)7697 mcrA Δ(mrr-hsdRMS-mcrBC) proBA::pir116 araE201 ΔrpoZ Δflu ΔcsgABCDEFG ΔpgaC λ–
                    </td>
                <td>addgene</td>
            </tr>
        </tbody></table>
        <p><strong>Table S2. The plasmids which were used in this study.</strong></p><br>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody><tr>
                <th>Names</th>
                <th>Relevant characteristics</th>
                <th>Purpose</th>
            </tr>

            <tr>
                <td>pQE-80L</td>
                <td>pBR322-KanR</td>
                <td>Expression vector and negative control</td>
            </tr>
            <tr>
                <td>pQE-80L-eGFP</td>
                <td>pQE-80L-P22-T-PfdeAR-eGFP</td>
                <td>positive control</td>
            </tr>
            <tr>
                <td>pQE-80L-fdeR</td>
                <td>pBR322-p22-fdeR-T-PfdeAR-eGFP</td>
                <td>Biosensor</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-A</td>
                <td>pBR322-p22-fdeR-Mutpre-L268W
                    </td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-B</td>
                <td>pBR322-p22-fdeR-Mutpre-H170W</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-C</td>
                <td>pBR322-p22-fdeR-Mutpre-H170Y</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-D</td>
                <td>pBR322-p22-fdeR-Mutpre-H170W+L268W</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>pQE-80L-p22-fdeR-Mutpre-E</td>
                <td>pBR322-p22-fdeR-Mutpre-H170Y+L268W</td>
                <td>Site-directed mutagenesis</td>
            </tr>

            <tr>
                <td>AP1</td>
                <td>pSC101Cm-T7-gⅢ-eGFP</td>
                <td>positive selection</td>
            </tr>
            <tr>
                <td>AP2-neg</td>
                <td>pSC101Cm-T7-gⅢ-neg-RFP</td>
                <td>negative selection</td>
            </tr>
            <tr>
                <td>DP6</td>
                <td>A plasmid expresses mutagenic genes</td>
                <td>Expression of mutagenic genes </td>
            </tr>

            <tr>
                <td>pBT137-C</td>
                <td>M13 carrier section 
                    with replica origin
                    </td>
                <td>Construction of selection phage</td>
            </tr>

            <tr>
                <td>pBT137-D</td>
                <td>M13 carrier fraction 
                    containing gⅢ
                    </td>
                <td>Construction of selection phage</td>
            </tr>
            <tr>
                <td>AP-P-Mid2-T7RNAP</td>
                <td>Contains Narigenin riboswitch Mid2 and T7RNAP</td>
                <td>Construction of selection phage</td>
            </tr>

            <tr>
                <td>AP-P-High2-T7RNAP</td>
                <td>Contains Narigenin riboswitch High2 and T7RNAP</td>
                <td>Construction of selection phage</td>
            </tr>

            <tr>
                <td>pJC175e</td>
                <td>Provides low copy of gⅢ</td>
                <td>Transform into S2060 competent cell to obtain S2208</td>
            </tr>
        </tbody></table>
        <p><strong>Table. S3 The primers which were used in this study.</strong></p><br>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody>
                <tr><th>No.</th>
                <th>Name</th>
                <th>Sequence (5’ -> 3’)</th>
                <th>Function</th></tr>
                
                <tr>
                <td>1</td>
                <td>H170Y-1-F</td>
                <td>5'-gcagacGTAccgttcgcggaagacctct-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>2</td>
                <td>SITE-1-R</td>
                <td>5'-cgcatatggtgcactctcagtaca-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>3</td>
                <td>H170Y-2-R</td>
                <td>5'-gaacggTACgtctgcgtggtctggcgc-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>4</td>
                <td>SITE-2-F</td>
                <td>5'-tgtactgagagtgcaccatatgcg-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>5</td>
                <td>L268W-1-F</td>
                <td>5'-ctcgccCCAcgacagcggactctccttgatc-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>6</td>
                <td>SITE-1-R</td>
                <td>5'-cgcatatggtgcactctcagtaca-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>7</td>
                <td>L268W-2-R</td>
                <td>5'-ctgtcgTGGggcgagatgcggcagatgat-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>8</td>
                <td>SITE-2-F</td>
                <td>5'-tgtactgagagtgcaccatatgcg-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>9</td>
                <td>H170W-1-F</td>
                <td>5'-gcagacCCAccgttcgcggaagacctct-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>10</td>
                <td>SITE-1-R</td>
                <td>5'-cgcatatggtgcactctcagtaca-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>11</td>
                <td>H170W-2-R</td>
                <td>5'-gaacggTGGgtctgcgtggtctggcgc-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>12</td>
                <td>SITE-2-F</td>
                <td>5'-tgtactgagagtgcaccatatgcg-3'</td>
                <td>Site-directed mutagenesis</td>
            </tr>
            <tr>
                <td>13</td>
                <td>AP2-neg- GⅢneg-F</td>
                <td>5'-TTAAGACTCCTTATTACGCAGTATGTT-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>14</td>
                <td>AP2-neg- GⅢneg-R</td>
                <td>5'-GGTGGCTCTTCCCAAATGGCTCAAGTCG-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>15</td>
                <td>RFP-F</td>
                <td>5'-TGCTAGAACTGGCATGATTAAGCACCGGTGGAGTGAC-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>16</td>
                <td>RFP-R</td>
                <td>5'-GTAATAAGGAGTCTTAAATGGCGAGTAGCGAAGACGT-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>17</td>
                <td>AP2-neg-vector-F</td>
                <td>5'-TTTGGGAAGAGCCACCACCGGAACCG-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>18</td>
                <td>AP2-neg-
                    vector-R
                    </td>
                <td>5'-TCATGCCAGTTCTAGCATAACCCC-3'</td>
                <td>Construction of AP2-neg</td>
            </tr>
            <tr>
                <td>19</td>
                <td>AP2-pos-vector-F</td>
                <td>5'-CTCACCATTTAAGACTCCTTATTACGCAGTATGTTAG-3'</td>
                <td>Construction of AP2-pos</td>
            </tr>
            <tr>
                <td>20</td>
                <td>AP2-pos-vector-R</td>
                <td>5'-ACAAGTAATCATGCCAGTTCTAGCATAACC-3'</td>
                <td>Construction of AP2-pos</td>
            </tr>
            <tr>
                <td>21</td>
                <td>AP2-pos-eGFP-F</td>
                <td>5'-AGTCTTAAATGGTGAGCAAGGGCGAGGAG-3'</td>
                <td>Construction of AP2-pos</td>
            </tr>
            <tr>
                <td>22</td>
                <td>AP2-pos-eGFP-R</td>
                <td>5'-TGGCATGATTACTTGTACAGCTCGTCCATGCC-3'</td>
                <td>Construction of AP2-pos</td>
            </tr>
            <tr>
                <td>23</td>
                <td>pQE-pathway-vector-F</td>
                <td>5'-TAAGGCAGTTATTGGTGCCCTTA-3'</td>
                <td>Construction of pQE-pathway</td>
            </tr>
            <tr>
                <td>24</td>
                <td>pQE-pathway-vector-R</td>
                <td>5'-TTCTCGAGGTGAAGACGAAAGG-3'</td>
                <td>Construction of pQE-pathway</td>
            </tr>
            <tr>
                <td>25</td>
                <td>pQE-pathway-fragment-F</td>
                <td>5'-TCTTCACCTCGAGAATCAGCAGATGCCTGGCAGC-3'</td>
                <td>Construction of pQE-pathway</td>
            </tr>
            <tr>
                <td>26</td>
                <td>pQE-pathway-fragment-R</td>
                <td>5'-CCAATAACTGCCTTAAAAAAATTACTTGTACAGCTCGTCCATGC-3'</td>
                <td>Construction of pQE-pathway</td>
            </tr>
            <tr>
                <td>27</td>
                <td>pQE-SplitC-RBS-F</td>
                <td>5'-ACTAGACTACACTAAAAAGAGGACTATTCATCCGCTGCATGGCCCC-3'</td>
                <td>Substitution of RBS of eGFP</td>
            </tr>
            <tr>
                <td>28</td>
                <td>pQE-SplitC-RBS-R</td>
                <td>5'-CGCATATGGTGCACTCTCAGTACAATCTG-3'</td>
                <td>Substitution of RBS of eGFP</td>
            </tr>
            <tr>
                <td>29</td>
                <td>pQE-SplitD-RBS-F</td>
                <td>5'-TGTACTGAGAGTGCACCATATGCG-3'</td>
                <td>Substitution of RBS of eGFP</td>
            </tr>
            <tr>
                <td>30</td>
                <td>pQE-SplitD-RBS-R</td>
                <td>5'-TCTTTTTAGTGTAGTCTAGTAATCCCGTTGATATTGATCAAATGGATTGTTTTG-3'</td>
                <td>Substitution of RBS of eGFP</td>
            </tr>
            <tr>
                <td>31</td>
                <td>Mid2-random-F</td>
                <td>5'-CGATGCTGCTTATCGAAGTCTGA-3'</td>
                <td>Nested PCR to obtain the fragment of Mid2</td>
            </tr>
            <tr>
                <td>32</td>
                <td>Mid2-random-R</td>
                <td>5'-TGCAGAACGCGCCTGTTTATC-3'</td>
                <td>Nested PCR to obtain the fragment of Mid2</td>
            </tr>
            <tr>
                <td>33</td>
                <td>High2-random-F</td>
                <td>5'-AGGCGCGTTCTGCATTAGC-3'</td>
                <td>Nested PCR to obtain the fragment of High2</td>
            </tr>
            <tr>
                <td>34</td>
                <td>High-random-R</td>
                <td>5'-GCTGTAACAAGTTGTCTCAGGTGTTC-3'</td>
                <td>Nested PCR to obtain the fragment of High2</td>
            </tr>
            <tr>
                <td>35</td>
                <td>overlap-Mid2&High2-F</td>
                <td>5'-TACTACATAAATCAAACAGGGACGACGATGACACGAGA-3'</td>
                <td> Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>36</td>
                <td>overlap-Mid2-R</td>
                <td>5'-GTTAATCGTGTTCATAGATGCTCCTTCACTCGCATT-3'</td>
                <td>Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>37</td>
                <td>overlap-High2-R</td>
                <td>5'-GTTAATCGTGTTCATAGATGCTCCTTATTCCCCCC-3'</td>
                <td> Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>38</td>
                <td>overlap-T7RNAP-F</td>
                <td>5'-AGGAGCATCTATGAACACGATTAACATCGCTAAGA-3'</td>
                <td> Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>39</td>
                <td>overlap-T7RNAP-R</td>
                <td>5'-GGCATGAGCTCTTCAGCCTTACGCGAACGCGAAGTCC-3'</td>
                <td> Overlap PCR to obtain Mid2-T7RNAP and High2-T7RNAP</td>
            </tr>
            <tr>
                <td>40</td>
                <td>overlap-AP-P-V-F</td>
                <td>5'-GTTCGCGTAAGGCTGAAGAGCTCATGCCAG-3'</td>
                <td>Construction of the plasmid of AP-P-Mid2-T7RNAP and AP-P-High2-T7RNAP</td>
            </tr>
            <tr>
                <td>41</td>
                <td>overlap-AP-P-V-R</td>
                <td>5'-GTCGTCCCTGTTTGATTTATGTAGTAGACTAGAAGAGC-3'</td>
                <td>Construction of the plasmid of AP-P-Mid2-T7RNAP and AP-P-High2-T7RNAP</td>
            </tr>
            <tr>
                <td>42</td>
                <td>ep-fdeR-F</td>
                <td>5'-ACCTCGAGAACGATGCTGCTTATCGAAGTCTGA-3'</td>
                <td>Error-prone PCR to obtain fdeR</td>
            </tr>
            <tr>
                <td>43</td>
                <td>ep-fdeR-R</td>
                <td>5'-CTCGAATTCGGAGGAAACAAAG-3'</td>
                <td>Error-prone PCR to obtain fdeR</td>
            </tr>
            <tr>
                <td>44</td>
                <td>ep-vec-F</td>
                <td>5'-TGTTTCCTCCGAATTCGAGGT-3'</td>
                <td>Error-prone PCR to obtain pQE-80L used for vector</td>
            </tr>
            <tr>
                <td>45</td>
                <td>ep-vec-R</td>
                <td>5'-CATCGTTCTCGAGGTTCGTGATACGCCTATTTTTATAGGTTA-3'</td>
                <td>Error-prone PCR to obtain pQE-80L used for vector</td>
            </tr>
        </tbody></table>
        <p><strong>Table S4. Hill parameters, operational ranges and noise parameter values of the constructed and characterised biosensors. </strong><br>  M: the maximum normalised fluorescent signal, a:the basal normalised fluorescent signal, n: Hill coefficient (cooperativity), KM:Hill constant (half-maximal Naringenin concentration) and k:proportional maximum response parameter. NA: Not applicable.</p>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody><tr>
                <th>Plasmid abbrev.</th>
                <th>a (a.u.)</th>
                <th>n</th>
                <th>Km (mg/L)</th>
                <th>k (a.u.)</th>
                <th>Detection range (mM)</th>
                <th>Noise</th>
            </tr>
            <tr>
                <td>WT(NG)</td>
                <td>177909.6±964134.6</td>
                <td>4.24±2.83</td>
                <td>0.1±0.0</td>
                <td>825±151</td>
                <td>0.030-0.300</td>
                <td>34.10%</td>
            </tr>
            <tr>
                <td>WT(LG)</td>
                <td>NA</td>
                <td>NA</td>
                <td>NA</td>
                <td>NA</td>
                <td>NA</td>
                <td>16.30%</td>
            </tr>
            <tr>
                <td>49-11(NG)</td>
                <td>4595.7±392763.3</td>
                <td>0.21±1.59</td>
                <td>1417031.5±1426850000</td>
                <td>103320±18779200</td>
                <td>0.000-0.061</td>
                <td>23.30%</td>
            </tr>
            <tr>
                <td>49-11(LG)</td>
                <td>294088±170384.2</td>
                <td>1.04±0.34</td>
                <td>0.3±0.1</td>
                <td>12423±2750</td>
                <td>0.100-0.800</td>
                <td>28.10%</td>
            </tr>
            <tr>
                <td>49-18(NG)</td>
                <td>-1.1±456564.4</td>
                <td>1.07±7.34</td>
                <td>0±0.2</td>
                <td>5376±1514</td>
                <td>0.000-0.057</td>
                <td>15.20%</td>
            </tr>
            <tr>
                <td>49-18(LG)</td>
                <td>72361.8±331886.8</td>
                <td>1.99±0.45</td>
                <td>0.2±0.0</td>
                <td>11635±1045</td>
                <td>0.050-0.600</td>
                <td>52.00%</td>
            </tr>
            <tr>
                <td>49-36(NG)</td>
                <td>2429.9±115739.4</td>
                <td>0.19±0.92</td>
                <td>3407163.3±2177050000</td>
                <td>97914±10160300</td>
                <td>0.000-0.040</td>
                <td>14.70%</td>
            </tr>
            <tr>
                <td>49-36(LG)</td>
                <td>29607.8±493213.6</td>
                <td>0.9±0.45</td>
                <td>0.9±1.5</td>
                <td>24539±20141</td>
                <td>0.000-1.000</td>
                <td>64.00%</td>
            </tr>
            <tr>
                <td>49-39(NG)</td>
                <td>10784.9±664018.3</td>
                <td>1.24±1.55</td>
                <td>0.1±0</td>
                <td>4846±1307</td>
                <td>0.000-0.400</td>
                <td>18.20%</td>
            </tr>
            <tr>
                <td>49-39(LG)</td>
                <td>2444392±782420</td>
                <td>0.4±0.49</td>
                <td>579206.3±476701000</td>
                <td>2790738±888273000</td>
                <td>0.000-1.000</td>
                <td>20.80%</td>
            </tr>
            <tr>
                <td>49-49(NG)</td>
                <td>1599.8±199867.9</td>
                <td>0.15±1.58</td>
                <td>2663307.6±2608130000</td>
                <td>53038±5764672</td>
                <td>0.000-0.025</td>
                <td>15.20%</td>
            </tr>
            <tr>
                <td>49-49(LG)</td>
                <td>215015.5±315889.9</td>
                <td>0.61±0.54</td>
                <td>95704.9±218666000</td>
                <td>10349200±14388500000</td>
                <td>0.000-1.000</td>
                <td>44.90%</td>
            </tr>
            <tr>
                <td>49-66(NG)</td>
                <td>NA</td>
                <td>NA</td>
                <td>NA</td>
                <td>NA</td>
                <td>NA</td>
                <td>7.60%</td>
            </tr>
            <tr>
                <td>49-66(LG)</td>
                <td>653795.8±190387.239</td>
                <td>0.7±0.4</td>
                <td>0.7±1.5</td>
                <td>8344±6522</td>
                <td>0.000-1.000</td>
                <td>22.10%</td>
            </tr>
            <tr>
                <td>49-70(NG)</td>
                <td>1.5±720004.9</td>
                <td>7.09±167.05</td>
                <td>0.1±0.4</td>
                <td>4161±765</td>
                <td>0.057-0.1300</td>
                <td>15.80%</td>
            </tr>
            <tr>
                <td>49-70(LG)</td>
                <td>2417977.2±1416681.1</td>
                <td>0.34±0.68</td>
                <td>308760.8±187508000</td>
                <td>521719±102500000</td>
                <td>0.000-1.000</td>
                <td>20.60%</td>
            </tr>
            <tr>
                <td>49-92(NG)</td>
                <td>NA</td>
                <td>NA</td>
                <td>NA</td>
                <td>NA</td>
                <td>NA</td>
                <td>21.00%</td>
            </tr>
            <tr>
                <td>49-92(LG)</td>
                <td>310381.4±257069.3</td>
                <td>1.34±0.44</td>
                <td>0.2±0</td>
                <td>10165±1421</td>
                <td>0.020-0.400</td>
                <td>26.40%</td>
            </tr>
        </tbody></table>
        <p><strong>Table S5. Supplementary sequence. </strong><br> The optimal PCR system ion environmental conditions were obtained by adjusting the amount of StarMut Enhancer. 0μL，2μL，4μL，6μL，8μL and 10μL before the sequence represented the amount of StarMut Enhancer added to the 20 μL PCR system respectively. When 4 μL of StarMut Enhancer was added, the results of fragment sequencing reflected the higher mutation rate and more evenly distributed of mutation sites. As a consequence this system was applied to our formal error-prone PCR. <br>
<strong>0μL:</strong><br>
            TCAGCAGATGCCTGGCAGCGCCGCATCCATCTCCTGCGCACTCTCCAGAAACACCCGACGCAGCCACTGGATGCCAGGATCATTGCTGC
GGTAGCGATGCCACTGCATCATCTGCCGCATCTCGCCCAGCGACAGCGGACTCTCCTTGATCACCACCGGCCATTGCGGAGCCAGCAGC
TGCGCCAGCCGGGCATGCACCGTGGCGATGCGGTCCGTCCCCTGTACCAGCGCCAGCGCAGAAGCGAAGCTGAAGCTGGTCACTTCCAC
CCGGCGCGCAAAGCCCAGCTTCCTGGCCATCCACGCCTCCACCGACGACGCATTGGCCCCAGGCGGCACCATCACCACATGGCCTGAGG
CCATGTAGCGTTCCAGCGTCAGCTCGCCTTGCGCCAGCGCACTGTCGCGCCAGACCACGCAGACATGCCGTTCGCGGAAGACCTCTTCGG
CAGGATGATCGGGCGTGCAGAATTCCTGCGGCAAGACCAGCAGGTCCACCTCGGCCCGATCCAGCGAGCGGGTCGGGTCTTGCACCTGC
GGCATCAGGGCAAAGCGGATGTGCTTGCCCTCGGCGTGCGCGCGCGCCAGCACCCGGGGGATGAGGACGGAGAGCGTAAAGTCCGAAA
CCGAGATGCGAAACTCGCGCGTGGACTCGGCCGGCACGAAGGCCGGCAGCGCCGCGATGGAGCCATCGATACGCCGCAGCACATCATG
CACCGCATCCTTGAGCACCTCGGCGCGCGGCGTGGGCTCCATGCGCCGGCCCACCTGGATCAGCAATTCATCATCGAAATACTCGCGCAG
CCGCGCCAGGGCATTGCTCATGGCCGACTGGGTCAGATGGATCTTTTCGG<strong style="color:red;">G</strong>GG<strong style="color:red;">G</strong>GCGGCTGATGCTCATCTCCGTGAGCAGTGCATCCAG
GGCGACCAGAAGATTGAGGTCGAGCTTGTTGAAACGCAT

<br>
            <strong>2μL:</strong><br> 
            TCAGCAGATGCCTGGCAGCGCCGCATCCAT<strong style="color:red;">T</strong>TCCTGCGCACTCTCCAGAAACACCCGACGCAGCCACTGGATGCCAGGATCATTGCTGCGG
TAGCGATGCCACTGCATCATCTGCCGCATCTCGCCCAGCGACAGCGGACTCTCCTTGATCACCACCGGCCATTGCGGAGCCAGCAGCTGCG
CCAGCCGGGCATGCACCGTGGCGATGCGGTCCGTCCCCTGTACCAGCGCCAGCGCAGAAGCGAAGCTGAAGCTGGTCACTTCCACCCGGC
GCGCAAAGCCCAGCTTCCTGGCCATCCACGCCTCCACCGACGACGCATTGGCCCCAGGCGGCACCATCACCACATGGCCTGAGGCCATGTA
GCGTTCCAGCGTCAGCTCGCCTTGCGCCAGCGCACTGTCGCGCCAGACCACGCAGACATGCCGTTCGCGGAAGACCTCTTCGGCAGGATGA
TCGGGCGTGCAGAATTCCTGCGGCAAGACCAGCAGGTCCACCTCGGCCCGATCCAGCGAGCGGGTCGGGTCTTGCACCTGCGGCATCAGGG
CAAAGCGGATGTGCTTGCCCTCGGCGTGCGCGCGCGCCAGCACCCGGGGGATGAGGACGGAGAGCGTAAAGTCCGAAACCGAGATGCGAA
ACTCGCGCGTGGACTCGGCCGGCACGAAGGCCGGCAGCGCCGCGATGGAGCCATCGATACGCCGCAGCACATCATGCACCGCATCCTTGAG
CACCTCGGCGCGCGGCGTGGGCTCCATGCGCCGGCCCACCTGGATCAGCAATTCATCATCGAAATACTCGCGCAGCCGCGCCAGGGCATTGC
TCATGGCCGACTGGCT<strong style="color:red;">T</strong>CAGATGGA<strong style="color:red;">A</strong>TCTTTTCGG<strong style="color:red;">G</strong>CGG<strong style="color:red;">G</strong>CGCGGCTGATGCTCATCTCCGTGAGCAGTGCATCCAGGGCGACCA<strong style="color:red;">A</strong>AA<strong style="color:red;">A</strong>ATT
GAGGTCGAGCTTGTTGAAACGCAT
<br>
            <strong>4μL:</strong><br> 
            TCAGCAGAT<strong style="color:red;">C</strong>CCTGGCAGCGCCGCATCCATCTCCTGCGCACTCTCCAGAAACACCCGACGCAGCCACTGGATGCCAGGATCATTGCTGCGGT
AGCGATGCCACTGCATCAT<strong style="color:red;">T</strong>CTGCCGCATCTCGCCCAGCGACAGC<strong style="color:red;">C</strong>GACTCTCCTTGATCACCACCGGCC<strong style="color:red;">C</strong>ATTGCGGAGCCAGCAGCTGCGCCA
GCCGGGCA<strong style="color:red;">A</strong>GCACCGTGGCGATGCGGTCCGTCCCC<strong style="color:red;">C</strong>TGTACCAGCGCCAGCGCAGAAGCGAAGCTGAAGCTGGTCACTTCCACCC<strong style="color:red;">C</strong>GCGCGCAA
AGCCCAGCTTCCTGGCCATCCACGGCCTCCACCGACGACGCATTGGCCCCAGGCGGCACCATCACCACATGGCCTGAGGCCATGTAGCGTTCCAG
CGTCAGCTCGCCTTGCGCCAGCGCACTGTCGCGCCAGACCACGCAGACATGCCGTTCGCGGAAGACCTCTTCGGCAGGATGATCGGGCGTGCAGA
ATTCCTGCGGCAAGACCAGCAGGTCCACCTCGGCCCGATCCAGCGAGCGGGTCGGGTCTTGCACCTGCGGCATCAGGGCAAAGCGGATGTGCTTG
CCCTCGGCGTGCGCGCGCGCCAGCACCCGGGGGATGAGGACGGAGAGCGTAAAGTCCGAAACCGAGATGCGAAACTCGCGCGTGGACTCGGCCG
GCACGAAGGCCGGCAGCGCCGCGATGGAGCCATCGATACGCCGCAGCACATCATGCACCGCATCCTTGAGCACCTCGGCGCGCGGCGTGGGCTCC
ATGCGCCGGCCCACCTGGATCAGCAATTCATCATCGAAATACTCGCGCAGCCGCGCCAGGGCATTGCTCATGGCCGACTGGCTCAGATGGATCTTT
TCGGCGGCGCGGCTGATGCTCATCTCCGTGAGCAGTGCATCCAGGGCGACCA<strong style="color:red;">A</strong>AAGATTGAGGTCGAGCTTGTTGAAACGCAT <br>
<strong>6μL:</strong><br> 
TCAGCAGATGCCTGGCAGCGCCGCATCCAT<strong style="color:red;">T</strong>TCCTG<strong style="color:red;">G</strong>GCA<strong style="color:red;">T</strong>TCTCCAGAAACACCCGACGCAGCCACTGGATGCCAGGATCATTGCTGCGGTA
GCGATGCCACTGCATCATCTGCCGCATCTCGCCCAGCGACAGCGGACTCTCCTTGATCACCACCGGCCATTGCGGAGCCAGCAGCTGCGCCAGCCG
GGCATGCACCGTGGCGATGCGGTCCGTCCCCTGTACCAGCGCCAGCGCAGAAGCGAAGCTGAAGCTGGTCACTTCCACCCGGCGCGCAAAGCCCA
GCTTCCTGGCCATCCACGCCTCCACCGACGACGCATTGGCCCCAGGCGGCACCATCACCACATGGCCTGAGGCCATGTAGCGTTCCAGCGTCAGCTC
GCCTTGCGCCAGCGCACTGTCGCGCCAGACCACGCAGACATGCCGTTCGCGGAAGACCTCTTCGGCAGGATGATCGGGCGTGCAGAATTCCTGCGG
CAAGACCAGCAGGTCCACCTCGGCCCGATCCAGCGAGCGGGTCGGGTCTTGCACCTGCGGCATCAGGGCAAAGCGGATGTGCTTGCCCTCGGCGTG
CGCGCGCGCCAGCACCCGGGGGATGAGGACGGAGAGCGTAAAGTCCGAAACCGAGATGCGAAACTCGCGCGTGGACTCGGCCGGCACGAAGGCCG
GCAGCGCCGCGATGGAGCCATCGATACGCCGCAGCACATCATGCACCGCATCCTTGAGCACCTCGGCGCGCGGCGTGGGCTCCATGCGCCGGCCCA
CCTGGATCAGCAATTCATCATCGAAATACTCGCGCAGCCGCGCCAGGGCATTGCTCATGGCCGACTGGCTCAGATGGATCTTTTCGGCGGCGCGGCT
GATGCTCATCTCCGTGAGCAGTGCATCCAGGGCGACCAGAAGATTGAGGTCGATCGAGCTTGTTGAAACGCAT <br>
<strong>8μL:</strong><br> 
TCAGCAGATGCCTGGCAGCGCCGCATCCATCTCCTGCGCACTCTCCAGAAACACCCGACGCAGCCACTGGATGCCAGGATCATTGCTGCGGTAG
CGATGCCACTGCATCATCTGCCGCATCTCGCCCAGCGACAGCGGACTCTCCTTGATCACCACCGGCCATTGCGGAGCCAGCAGCTGCGCCAGCCGGG
CATGCACCGTGGCGATGCGGTCCGTCCCCTGTACCAGCGCCAGCGCAGAAGCGAAGCTGAAGCTGGTCACTTCCACCCGGCGCGCAAAGCCCAGCTT
CCTGGCCATCCACGCCTCCACCGACGACGCATTGGCCCCAGGCGGCACCATCACCACATGGCCTGAGGCCATGTAGCGTTCCAGCGTCAGCTCGCCT
TGCGCCAGCGCACTGTCGCGCCAGACCACGCAGACATGCCGTTCGCGGAAGACCTCTTCGGCAGGATGATCGGGCGTGCAGAATTCCTGCGGCAAGA
CCAGCAGGTCCACCTCGGCCCGATCCAGCGAGCGGGTCGGGTCTTGCACCTGCGGCATCAGGGCAAAGCGGATGTGCTTGCCCTCGGCGTGCGCGCG
CGCCAGCACCCGGGGGATGAGGACGGAGAGCGTAAAGTCCGAAACCGAGATGCGAAACTCGCGCGTGGACTCGGCCGGCACGAAGGCCGGCAGCGC
CGCGATGGAGCCATCGATACGCCGCAGCACATCATGCACCGCATCCTTGAGCACCTCGGCGCGCGGCGTGGGCTCCATGCGCCGGCCCACCTGGATCA
GCAATTCATCATCGAAATACTCGCGCAGCCGCGCCAGGGCATTGCTCATGGCCGACTGGCTCAGATGGATCTTTTCGGCGGCGCGGCTGATGCTCATC
TCCGTGAGCAGTGCATCCAGGGCGACCAGAAGATTGAGGTCGAGCTTGTTGAAACGCAT <br>
<strong>10μL:</strong><br> 
TCAGCAGATGCCTGGCAGCGCCGCATCCATCTCCTGCGCACTCTCCAGAAACACCCGACGCAGCCACTGGATGCCAGGATCATTGCTGCGGTAG
CGATGCCACTGCATCATCTGCCGCATCTCGCCCAGCGACAGCGGACTCTCCTTGATCACCACCGGCCATTGCGGAGCCAGCAGCTGCGCCAGCCGGGC
ATGCACCGTGGCGATGCGGTCCGTCCCCTGTACCAGCGCCAGCGCAGAAGCGAAGCTGAAGCTGGTCACTTCCACCCGGCGCGCAAAGCCCAGCTTCC
TGGCCATCCACGCCTCCACCGACGACGCATTGGCCCCAGGCGGCACCATCACCACATGGCCTGAGGCCATGTAGCGTTCCAGCGTCAGCTCGCCTTGCG
CCAGCGCACTGTCGCGCCAGACCACGCAGACATGCCGTTCGCGGAAGACCTCTTCGGCAGGATGATCGGGCGTGCAGAATTCCTGCGGCAAGACCAGC
AGGTCCACCTCGGCCCGATCCAGCGAGCGGGTCGGGTCTTGCACCTGCGGCATCAGGGCAAAGCGGATGTGCTTGCCCTCGGCGTGCGCGCGCGCCAGC
ACCCGGGGGATGAGGACGGAGAGCGTAAAGTCCGAAACCGAGATGCGAAACTCGCGCGTGGACTCGGCCGGCACGAAGGCCGGCAGCGCCGCGATGG
AGCCATCGATACGCCGCAGCACATCATGCACCGCATCCTTGAGCACCTCGGCGCGCGGCGTGGGCTCCATGCGCCGGCCCACCTGGATCAGCAATTCAT
CATCGAAATACTCGCGCAGCCGCGCCAGGGCATTGCTCATGGCCGACTGGCTCAGATGGATCTTTTCGGCGGCGCGGCTGATGCTCATCTCCGTGAGCA
GTGCATCCAGGGCGACCA<strong style="color:red;">A</strong>AAGATTGAGGTCGAGCTTGTTGAAACGCAT
 <br>

        </p>
        <p><strong>Table S6. DNA sequence of liquiritigenin biosensor</strong><br> Annotated nucleotide sequence of the complete pQE-80L-fdeR plasmid constructed in this study. FdeR is highlighted in purple, the RBS in orange, the Kozak sequence in cornflower blue, the P22 core promoter in red, the rrnB T1 terminator in grey, PfdeAR in yellow, eGFP in green.<br>
            <strong style="color: purple;">Tcagcagatgcctggcagcgccgcatccatctcctgcgcactctccagaaacacccgacgcagccactggatgccaggatcattgctgcggtagcgatgccactgcatcatctgccgcatctcgcccagcgacagcg
                gactctccttgatcaccaccggccattgcggagccagcagctgcgccagCcgggcatgcaccgtggcgatgcggtccgtcccctgtaccagcgccagcgcagaagcgaagctgaagctggtcacttccacccggcg
                cgcaaagcccagcttcctggccatccacgcctccaccgacgacgcattggccccaggcggcaccatcaccacatggcctgaggccatgtagcgttccaGcgtcagctcgccttgcgccagcgcactgtcgcgccaga
                ccacgcagacatgccgttcgcggaagacctcttcggcaggatgatcgggcgtgcagaattcctgcggcaagaccagcaggtccacctcggcccgatccagcgagcgggtcgggtcttgcacctgcggcatcagggca
                aagcggatgTgcttgccctcggcgtgcgcgcgcgccagcacccgggggatgaggacggagagcgtaaagtccgaaaccgagatgcgaaactcgcgcgtggactcggccggcacgaaggccggcagcgccgcgat
                ggagccatcgatacgccgcagcacatcatgcaccgcatccttgagcacctcggcgcgcgGcgtgggctccatgcgccggcccacctggatcagcaattcatcatcgaaatactcgcgcagccgcgccagggcattgct
                catggccgactggctcagatggatcttttcggcggcgcggctgatgctcatctccgtgagcagtgcatccagggcgaccagaagattgaggtcgagcttgttgaaacgcat</strong><strong style="color: black;">ctttgttt</strong><strong style="color: orange;">cctccg</strong><strong style="color: black;">aattcgaggtcgacgg
                    atcccaagcttcttctagagcg</strong><strong style="color: rgb(24, 144, 241);">gccgccatgg</strong><strong style="color: rgb(255, 0, 0);">ATATGTACACATTATATCACATCTATTCCAAAATGTCAA</strong><strong style="color: black;">ctgaaggcgctcctcatactggctgaatacatgacccgacacgttatca
                        gaatttatcatgtcatgattgcttacaatactcacataattgtgaaaattgtagaagccgcaacattcaattatcaatcaaaatcatgtccgcgccagtttcgttcctcgccgttgcactaatacagggtaattccctcaccac
                        ataaggaattaatgacgtcaggtggcacttttccacagcactaatacagggtaattccctcaccacataaggacag</strong><strong style="color: rgb(124, 123, 123);">gagagcgttcaccgacaaacaacagataaaacgaaaggcccagtctttcgactgagcctttcgt
                            tttatttg</strong><strong style="color: rgb(0, 0, 0);">atgcctggcagttccccaatcagtcggaacggcaatgctagcccatggtta</strong><strong style="color: rgb(217, 255, 0);">ggcgctggtctccgttgttgtgcttgttcttgccgaccctcggatagacgacggatggggtggtcaatgtattgatgccgtccata
                                tcatgaatcaaaacaatccatttgatcaatatcaagctcactcttaagcttcactTcatccgctgcatggccccaccagaaagggctggcgcggcaagccggcggcgcactcgcactggatgcgccgctgttgagcctggcc
                                atgacaacgcgccgatagcggccacaccccgccaggcagggtaggagacaaggagacagac</strong><strong style="color: rgb(3, 175, 11);">atggtgagcaagggcgaggagctgttcaccggggtggtgcccatcctggtcgagctggacggcgacgtaaacggcca
                                    caagttcagcgtgtccggcgagggcgagggcgatgccacctacggcaagctgaccctgaagttcatctgcaccaccggcaagctgcccgtgccctggcccaccctcgtgaccaccctgacctacggcgtgcagtgcttcagc
                                    cgctaccccgaccacatgaagcagcacgacttcttcaagtccgccatgcccgaaggctacgtccaggagcgcaccatcttcttcaaggacgacggcaactacaagacccgcgccgaggtgaagttcgagggcgacaccctg
                                    gtgaaccgcatcgagctgaagggcatcgacttcaaggaggacggcaacatcctggggcacaagctggagtacaactacaacagccacaacgtctatatcatggccgacaagcagaagaacggcatcaaggtgaacttcaa
                                    gatccgccacaacatcgaggacggcagcgtgcagctcgccgaccactaccagcagaacacccccatcggcgacggccccgtgctgctgcccgacaaccactacctgagcacccagtccgccctgagcaaagaccccaacg
                                    agaagcgcgatcacatggtcctgctggagttcgtgaccgccgccgggatcactctcggcatggacgagctgtacaag</strong><strong style="color: rgb(0, 0, 0);">taa</strong>
        <p><strong>Table S7. DNA sequences of  riboswiches.</strong></p>
        <table style="border:black 2px solid;text-align: center;align-items: center;margin: 0 auto;">
            <tbody><tr>
                <th>Type of riboswitch</th>
                <th>Sequence of riboswitches</th>
            </tr>
            <tr>
                <td>Mid2</td>
                <td>gggacgacgatgacacgagatgccaaagggcagtactagcagttgtggtagagtcaccacaaccatatggcatacgccgatcacatgaccaaatgcgagtgaaggagcatct                </td>
            </tr>
            <tr><td>High-2</td>
            <td>gggacgacgatgacacgagatgccaaagggcagtactagcagttgtggtagagtcaccacaaccatatggcatacgccgatcacatgaccagggggggaataaggagcatct</td></tr>
        </tbody></table>




                                </div>
                                <div style="height: 100px;"></div>
   
</div>

<div class="side sidesup">
    <ul class="sidelist">
        <li class="si"><a href="#b"><h1>FIGURE</h1></a></li>
        <li class="si"><a href="#p"><h1>TABLE</h1></a></li>
    </ul>
   </div>


    <!-- footer -->
    <section class="footer fs">
        <div class="grid-wrapper">
            <div class="footer1">
                <h1 class="private">Tech Bees</h1>
                <p class="bottom">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Est ab beatae itaque ipsam
                    corporis maxime
                    saepe, voluptatum sapiente veritatis repellat!</p>
                <div class="social-profiles">
                    <i class="fa fa-facebook"></i>
                    <i class="fa fa-twitter"></i>
                    <i class="fa fa-instagram"></i>
                    <i class="fa fa-linkedin"></i>
                </div>
            </div>
            <div class="col3 footer2 fleft">
                <h1>Contact Info</h1>
                <h4>Lorem ipsum dolor sit.</h4>
                <h4>Lorem ipsum dolor sit.</h4>
                <h4>412616516</h4>
                <h4>@kjfhsdakf</h4>
                <h4>website.com</h4>
            </div>
            <div class="col3 footer3 fmid">
                <h1>Contact Us</h1>
                <h4>Company</h4>
                <h4>Events</h4>
                <h4>Projects</h4>
                <h4>Awards</h4>
                <h4>Open sources</h4>
            </div>
            <div class="col3 footer4">
                <h1>Custom Software</h1>
                <h4>Company</h4>
                <h4>Events</h4>
                <h4>Projects</h4>
                <h4>Awards</h4>
                <h4>Open sources</h4>
            </div>
        </div>
        <hr>
        <div class="copyright">
            <h4>Copyright Hyc</h4>
            <h4>Teams & Members</h4>
            <h4>Privacy</h4>
        </div>
        <!-- scroll to top btn -->
        <i id="scrolltp" class="fa fa-arrow-up"></i>
    </section>

    <!-- 导航栏二级 -->
    <script>

        var nav = document.querySelectorAll('.second');
        for (var i = 0; i < nav.length; i++) {
            nav[i].addEventListener('mouseenter', function () {
                var first = this.querySelector('.first');
                var secondChild = first.children[1];
                var img = this.querySelector('.ims');

                secondChild.style.opacity = "0";
                secondChild.style.transition = "opacity 0.5s";
                secondChild.style.display = "block";

                setTimeout(function () {
                    secondChild.style.opacity = "1";
                }, 200);

                img.classList.add("fade-out");

                img.addEventListener("transitionend", function () {
                    img.src = "../img/icon2.png";
                    img.classList.remove("fade-out");
                }, { once: true });
            });

            nav[i].addEventListener('mouseleave', function () {
                var first = this.querySelector('.first');
                var secondChild = first.children[1];
                var img = this.querySelector('.ims');

                secondChild.style.opacity = "0";
                secondChild.style.transition = "opacity 0.3s";

                img.classList.add("fade-out");

                img.addEventListener("transitionend", function () {
                    secondChild.style.display = "none";
                    img.src = "../img/icon1.png";
                    img.classList.remove("fade-out");
                }, { once: true });
            });
        }


    </script>
    <!-- normal -->
    <script src="../javascript/p_lbt.js"></script>

    <!-- AOS -->
    <script src="https://unpkg.com/aos@next/dist/aos.js"></script>
    <script>
       
    </script>
    <!-- App . js 图片轮播 -->
    <script src="../javascript/p_app.js"></script>

    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"></script>
</body>

</html>